<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="Dina Schuster" />
  <meta name="dcterms.date" content="2025-05-21" />
  <meta name="keywords" content="markdown, publishing, manubot" />
  <title>Advances in Mass Spectrometry for Membrane Protein Pharmacology</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    div.columns{display: flex; gap: min(4vw, 1.5em);}
    div.column{flex: auto; overflow-x: auto;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    ul.task-list li input[type="checkbox"] {
      width: 0.8em;
      margin: 0 0.8em 0.2em -1.6em;
      vertical-align: middle;
    }
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
    }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
  <!--
  Manubot generated metadata rendered from header-includes-template.html.
  Suggest improvements at https://github.com/manubot/manubot/blob/main/manubot/process/header-includes-template.html
  -->
  <meta name="dc.format" content="text/html" />
  <meta property="og:type" content="article" />
  <meta name="dc.title" content="Advances in Mass Spectrometry for Membrane Protein Pharmacology" />
  <meta name="citation_title" content="Advances in Mass Spectrometry for Membrane Protein Pharmacology" />
  <meta property="og:title" content="Advances in Mass Spectrometry for Membrane Protein Pharmacology" />
  <meta property="twitter:title" content="Advances in Mass Spectrometry for Membrane Protein Pharmacology" />
  <meta name="dc.date" content="2025-05-21" />
  <meta name="citation_publication_date" content="2025-05-21" />
  <meta property="article:published_time" content="2025-05-21" />
  <meta name="dc.modified" content="2025-05-21T23:22:39+00:00" />
  <meta property="article:modified_time" content="2025-05-21T23:22:39+00:00" />
  <meta name="dc.language" content="en-US" />
  <meta name="citation_language" content="en-US" />
  <meta name="dc.relation.ispartof" content="Manubot" />
  <meta name="dc.publisher" content="Manubot" />
  <meta name="citation_journal_title" content="Manubot" />
  <meta name="citation_technical_report_institution" content="Manubot" />
  <meta name="citation_author" content="Dina Schuster" />
  <meta name="citation_author_institution" content="Sarafan ChEM-H, Stanford University, Stanford, CA, 94305, USA" />
  <meta name="citation_author_orcid" content="0000-0001-6611-8237" />
  <link rel="canonical" href="https://dschust-r.github.io/review_ms_mem_prot/" />
  <meta property="og:url" content="https://dschust-r.github.io/review_ms_mem_prot/" />
  <meta property="twitter:url" content="https://dschust-r.github.io/review_ms_mem_prot/" />
  <meta name="citation_fulltext_html_url" content="https://dschust-r.github.io/review_ms_mem_prot/" />
  <meta name="citation_pdf_url" content="https://dschust-r.github.io/review_ms_mem_prot/manuscript.pdf" />
  <link rel="alternate" type="application/pdf" href="https://dschust-r.github.io/review_ms_mem_prot/manuscript.pdf" />
  <link rel="alternate" type="text/html" href="https://dschust-r.github.io/review_ms_mem_prot/v/6d5fabdb5378db0eb24e65189ed23ea0c3e3fe58/" />
  <meta name="manubot_html_url_versioned" content="https://dschust-r.github.io/review_ms_mem_prot/v/6d5fabdb5378db0eb24e65189ed23ea0c3e3fe58/" />
  <meta name="manubot_pdf_url_versioned" content="https://dschust-r.github.io/review_ms_mem_prot/v/6d5fabdb5378db0eb24e65189ed23ea0c3e3fe58/manuscript.pdf" />
  <meta property="og:type" content="article" />
  <meta property="twitter:card" content="summary_large_image" />
  <link rel="icon" type="image/png" sizes="192x192" href="https://manubot.org/favicon-192x192.png" />
  <link rel="mask-icon" href="https://manubot.org/safari-pinned-tab.svg" color="#ad1457" />
  <meta name="theme-color" content="#ad1457" />
  <!-- end Manubot generated metadata -->
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
</head>
<body>
<header id="title-block-header">
<h1 class="title">Advances in Mass Spectrometry for Membrane Protein Pharmacology</h1>
</header>
<p><small><em>
This manuscript
(<a href="https://dschust-r.github.io/review_ms_mem_prot/v/6d5fabdb5378db0eb24e65189ed23ea0c3e3fe58/">permalink</a>)
was automatically generated
from <a href="https://github.com/dschust-r/review_ms_mem_prot/tree/6d5fabdb5378db0eb24e65189ed23ea0c3e3fe58">dschust-r/review_ms_mem_prot@6d5fabd</a>
on May 21, 2025.
</em></small></p>
<h2 id="authors">Authors</h2>
<ul>
<li><strong>Dina Schuster</strong>
<sup><a href="#correspondence">✉</a></sup><br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-6611-8237">0000-0001-6611-8237</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/dschust-r">dschust-r</a>
<br>
<small>
Sarafan ChEM-H, Stanford University, Stanford, CA, 94305, USA
</small></li>
</ul>
<div id="correspondence">
<p>✉ — Correspondence possible via <a href="https://github.com/dschust-r/review_ms_mem_prot/issues">GitHub Issues</a>
or email to
Dina Schuster &lt;dschust@stanford.edu&gt;.</p>
</div>
<h2 class="page_break_before" id="abstract">Abstract</h2>
<p>Membrane proteins are important targets for pharmacological research, due to their involvement in various cellular processes, ranging from signal transduction to transport, and cellular communication. While they make up roughly 30% of the human proteome and are the target of more than half of all approved drug products, their hydrophobicity and low abundance pose significant challenges for drug discovery and development.
Mass spectrometry has emerged as a powerful tool for studying membrane protein structure, dynamics, regulation, as well as their interaction with proteins, small molecules, and lipids. In this review, we summarize recent findings and methodological advances that facilitate the study of cell surface organization, the discovery of druggable membrane proteins, and the identification of binding sites.</p>
<h2 class="page_break_before" id="novel-mass-spectrometric-methods-for-the-identification-of-drug-targets-and-binding-sites">1. Novel mass spectrometric methods for the identification of drug targets and binding sites</h2>
<p><span class="citation" data-cites="XUv5zqVu AVhHIfFo umwkwUjn 1AK6I4gVP 1F3LPvBP2 1c0XVRC1 G8GI9D01 eQaE8YUq 1DCAgfaxv">[<a href="#ref-XUv5zqVu" role="doc-biblioref">1</a>,<a href="#ref-AVhHIfFo" role="doc-biblioref">2</a>,<a href="#ref-umwkwUjn" role="doc-biblioref">3</a>,<a href="#ref-1AK6I4gVP" role="doc-biblioref">4</a>,<a href="#ref-1F3LPvBP2" role="doc-biblioref">5</a>,<a href="#ref-1c0XVRC1" role="doc-biblioref">6</a>,<a href="#ref-G8GI9D01" role="doc-biblioref">7</a>,<a href="#ref-eQaE8YUq" role="doc-biblioref">8</a>,<a href="#ref-1DCAgfaxv" role="doc-biblioref">9</a>]</span></p>
<h3 id="accessibility-approaches">Accessibility approaches</h3>
<p><span class="citation" data-cites="1AK6I4gVP 1F3LPvBP2 3sI32rB2 UETOKcpN cYi0OSgq HhtMYgJS 12oF7PyjF">[<a href="#ref-1AK6I4gVP" role="doc-biblioref">4</a>,<a href="#ref-1F3LPvBP2" role="doc-biblioref">5</a>,<a href="#ref-3sI32rB2" role="doc-biblioref">10</a>,<a href="#ref-UETOKcpN" role="doc-biblioref">11</a>,<a href="#ref-cYi0OSgq" role="doc-biblioref">12</a>,<a href="#ref-HhtMYgJS" role="doc-biblioref">13</a>,<a href="#ref-12oF7PyjF" role="doc-biblioref">14</a>]</span></p>
<h3 id="thermal-proteome-profiling">Thermal proteome profiling</h3>
<p><span class="citation" data-cites="umwkwUjn 12LdlQcg6">[<a href="#ref-umwkwUjn" role="doc-biblioref">3</a>,<a href="#ref-12LdlQcg6" role="doc-biblioref">15</a>]</span></p>
<h3 id="proximity-based-approaches-and-chemical-cross-linking">Proximity-based approaches and chemical cross-linking</h3>
<p><span class="citation" data-cites="KSJWpyaf ixF12fLE 1083OUMws 1DXKD97TB 1A0ELIyuD uJCnPykT 5zsiKMpi wqOAIazw">[<a href="#ref-KSJWpyaf" role="doc-biblioref">16</a>,<a href="#ref-ixF12fLE" role="doc-biblioref">17</a>,<a href="#ref-1083OUMws" role="doc-biblioref">18</a>,<a href="#ref-1DXKD97TB" role="doc-biblioref">19</a>,<a href="#ref-1A0ELIyuD" role="doc-biblioref">20</a>,<a href="#ref-uJCnPykT" role="doc-biblioref">21</a>,<a href="#ref-5zsiKMpi" role="doc-biblioref">22</a>,<a href="#ref-wqOAIazw" role="doc-biblioref">23</a>]</span></p>
<h3 id="native-and-intact-mass-spectrometry-of-membrane-proteins">Native and intact mass spectrometry of membrane proteins</h3>
<p><span class="citation" data-cites="QM5L157g ehunITJ2 1AiSIssJj 17kjm5RCK 1mGBvUhI 19wOhPBGo my3tj08K 2W5sjAvY 1AcwgRB7T dNXuiXTr">[<a href="#ref-QM5L157g" role="doc-biblioref">24</a>,<a href="#ref-ehunITJ2" role="doc-biblioref">25</a>,<a href="#ref-1AiSIssJj" role="doc-biblioref">26</a>,<a href="#ref-17kjm5RCK" role="doc-biblioref">27</a>,<a href="#ref-1mGBvUhI" role="doc-biblioref">28</a>,<a href="#ref-19wOhPBGo" role="doc-biblioref">29</a>,<a href="#ref-my3tj08K" role="doc-biblioref">30</a>,<a href="#ref-2W5sjAvY" role="doc-biblioref">31</a>,<a href="#ref-1AcwgRB7T" role="doc-biblioref">32</a>,<a href="#ref-dNXuiXTr" role="doc-biblioref">33</a>]</span></p>
<h2 class="page_break_before" id="selective-probing-of-membrane-proteins-with-chemoproteomics">2. Selective probing of membrane proteins with chemoproteomics</h2>
<p><span class="citation" data-cites="jPUY8z0i KwvCLBXC jNq0LZ7n 8tpRy8vW 5qPERR5E ClHzzrGn">[<a href="#ref-jPUY8z0i" role="doc-biblioref">34</a>,<a href="#ref-KwvCLBXC" role="doc-biblioref">35</a>,<a href="#ref-jNq0LZ7n" role="doc-biblioref">36</a>,<a href="#ref-8tpRy8vW" role="doc-biblioref">37</a>,<a href="#ref-5qPERR5E" role="doc-biblioref">38</a>,<a href="#ref-ClHzzrGn" role="doc-biblioref">39</a>]</span></p>
<h2 class="page_break_before" id="mass-spectrometry-reveals-the-cell-surface-organization">3. Mass spectrometry reveals the cell surface organization</h2>
<p><span class="citation" data-cites="AVhHIfFo umwkwUjn 1083OUMws 1DXKD97TB QM5L157g KwvCLBXC laLpQc2u HnAihA6J">[<a href="#ref-AVhHIfFo" role="doc-biblioref">2</a>,<a href="#ref-umwkwUjn" role="doc-biblioref">3</a>,<a href="#ref-1083OUMws" role="doc-biblioref">18</a>,<a href="#ref-1DXKD97TB" role="doc-biblioref">19</a>,<a href="#ref-QM5L157g" role="doc-biblioref">24</a>,<a href="#ref-KwvCLBXC" role="doc-biblioref">35</a>,<a href="#ref-laLpQc2u" role="doc-biblioref">40</a>,<a href="#ref-HnAihA6J" role="doc-biblioref">41</a>]</span></p>
<h2 class="page_break_before" id="insights-into-g-protein-coupled-receptors">4. Insights into G protein-coupled receptors</h2>
<p><span class="citation" data-cites="umwkwUjn G8GI9D01 QM5L157g 17kjm5RCK 19wOhPBGo my3tj08K 1AcwgRB7T dNXuiXTr keHzgRUT">[<a href="#ref-umwkwUjn" role="doc-biblioref">3</a>,<a href="#ref-G8GI9D01" role="doc-biblioref">7</a>,<a href="#ref-QM5L157g" role="doc-biblioref">24</a>,<a href="#ref-17kjm5RCK" role="doc-biblioref">27</a>,<a href="#ref-19wOhPBGo" role="doc-biblioref">29</a>,<a href="#ref-my3tj08K" role="doc-biblioref">30</a>,<a href="#ref-1AcwgRB7T" role="doc-biblioref">32</a>,<a href="#ref-dNXuiXTr" role="doc-biblioref">33</a>,<a href="#ref-keHzgRUT" role="doc-biblioref">42</a>]</span></p>
<h2 class="page_break_before" id="remaining-challenges-and-outlook">5. Remaining challenges and outlook</h2>
<p><span class="citation" data-cites="eQaE8YUq dNXuiXTr eXw91M0D OuTfCeL4 Mak2jJSG">[<a href="#ref-eQaE8YUq" role="doc-biblioref">8</a>,<a href="#ref-dNXuiXTr" role="doc-biblioref">33</a>,<a href="#ref-eXw91M0D" role="doc-biblioref">43</a>,<a href="#ref-OuTfCeL4" role="doc-biblioref">44</a>,<a href="#ref-Mak2jJSG" role="doc-biblioref">45</a>]</span></p>
<h2 class="page_break_before" id="references">References</h2>
<!-- Explicitly insert bibliography here -->
<div id="refs" class="references csl-bib-body" role="doc-bibliography">
<div id="ref-XUv5zqVu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1. </div><div class="csl-right-inline"><strong>Capture of the Mouse Organ Membrane Proteome Specificity in Peptidisc Libraries</strong> <div class="csl-block">Frank Antony, Zora Brough, Zhiyu Zhao, Franck Duong van Hoa</div> <em>Journal of Proteome Research</em> (2024-01-17) <a href="https://doi.org/g5w3hs">https://doi.org/g5w3hs</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.3c00825">10.1021/acs.jproteome.3c00825</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/38232390">38232390</a></div></div>
</div>
<div id="ref-AVhHIfFo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2. </div><div class="csl-right-inline"><strong>A Peptidisc-Based Survey of the Plasma Membrane Proteome of a Mammalian Cell</strong> <div class="csl-block">Zhiyu Zhao, Arshdeep Khurana, Frank Antony, John W Young, Keeley G Hewton, Zora Brough, Tianshuang Zhong, Seth J Parker, Franck Duong van Hoa</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2023-08) <a href="https://doi.org/g9ks7s">https://doi.org/g9ks7s</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.mcpro.2023.100588">10.1016/j.mcpro.2023.100588</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/37295717">37295717</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10416069">PMC10416069</a></div></div>
</div>
<div id="ref-umwkwUjn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">3. </div><div class="csl-right-inline"><strong>Cell surface thermal proteome profiling tracks perturbations and drug targets on the plasma membrane</strong> <div class="csl-block">Mathias Kalxdorf, Ina Günthner, Isabelle Becher, Nils Kurzawa, Sascha Knecht, Mikhail M Savitski, HChristian Eberl, Marcus Bantscheff</div> <em>Nature Methods</em> (2021-01) <a href="https://doi.org/ghswc8">https://doi.org/ghswc8</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41592-020-01022-1">10.1038/s41592-020-01022-1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33398190">33398190</a></div></div>
</div>
<div id="ref-1AK6I4gVP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">4. </div><div class="csl-right-inline"><strong>Target Deconvolution by Limited Proteolysis Coupled to Mass Spectrometry</strong> <div class="csl-block">Viviane Reber, Matthias Gstaiger</div> <em>Methods in Molecular Biology</em> (2023) <a href="https://doi.org/g9ks7q">https://doi.org/g9ks7q</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/978-1-0716-3397-7_13">10.1007/978-1-0716-3397-7_13</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/37558949">37558949</a></div></div>
</div>
<div id="ref-1F3LPvBP2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">5. </div><div class="csl-right-inline"><strong>High-throughput peptide-centric local stability assay extends protein-ligand identification to membrane proteins, tissues, and bacteria</strong> <div class="csl-block">Kejia Li, Clement M Potel, Isabelle Becher, Nico Hüttmann, Martin Garrido-Rodriguez, Jennifer Schwarz, Mikhail M Savitski</div> <em>Cold Spring Harbor Laboratory</em> (2025-04-29) <a href="https://doi.org/g9ks7v">https://doi.org/g9ks7v</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/2025.04.28.650974">10.1101/2025.04.28.650974</a></div></div>
</div>
<div id="ref-1c0XVRC1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">6. </div><div class="csl-right-inline"><strong>Effects of theophylline on ADCY5 activation—From cellular studies to improved therapeutic options for ADCY5-related dyskinesia patients</strong> <div class="csl-block">Dirk Tänzler, Marc Kipping, Marcell Lederer, Wiebke F Günther, Christian Arlt, Stefan Hüttelmaier, Andreas Merkenschlager, Andrea Sinz</div> <em>PLOS ONE</em> (2023-03-03) <a href="https://doi.org/grv9ht">https://doi.org/grv9ht</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1371/journal.pone.0282593">10.1371/journal.pone.0282593</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/36867608">36867608</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9983822">PMC9983822</a></div></div>
</div>
<div id="ref-G8GI9D01" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">7. </div><div class="csl-right-inline"><strong>Phosphorylation Sites of the Gastric Inhibitory Polypeptide Receptor (GIPR) Revealed by Trapped-Ion-Mobility Spectrometry Coupled to Time-of-Flight Mass Spectrometry (TIMS-TOF MS)</strong> <div class="csl-block">Kyle A Brown, Rylie K Morris, Samantha J Eckhardt, Ying Ge, Samuel H Gellman</div> <em>Journal of the American Chemical Society</em> (2023-12-13) <a href="https://doi.org/g9ks7t">https://doi.org/g9ks7t</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/jacs.3c09078">10.1021/jacs.3c09078</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/38091482">38091482</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10842860">PMC10842860</a></div></div>
</div>
<div id="ref-eQaE8YUq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">8. </div><div class="csl-right-inline"><strong>Integrative structural modeling of macromolecular complexes using Assembline</strong> <div class="csl-block">Vasileios Rantos, Kai Karius, Jan Kosinski</div> <em>Nature Protocols</em> (2021-11-29) <a href="https://doi.org/gq586q">https://doi.org/gq586q</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41596-021-00640-z">10.1038/s41596-021-00640-z</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34845384">34845384</a></div></div>
</div>
<div id="ref-1DCAgfaxv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">9. </div><div class="csl-right-inline"><strong>Simple But Efficacious Enrichment of Integral Membrane Proteins and Their Interactions for In-Depth Membrane Proteomics</strong> <div class="csl-block">Pornparn Kongpracha, Pattama Wiriyasermkul, Noriyoshi Isozumi, Satomi Moriyama, Yoshikatsu Kanai, Shushi Nagamori</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2022-05) <a href="https://doi.org/g9ks7r">https://doi.org/g9ks7r</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.mcpro.2022.100206">10.1016/j.mcpro.2022.100206</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/35085786">35085786</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9062332">PMC9062332</a></div></div>
</div>
<div id="ref-3sI32rB2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">10. </div><div class="csl-right-inline"><strong>Structure of Mycobacterium tuberculosis Cya, an evolutionary ancestor of the mammalian membrane adenylyl cyclases</strong> <div class="csl-block">Ved Mehta, Basavraj Khanppnavar, Dina Schuster, Ilayda Kantarci, Irene Vercellino, Angela Kosturanova, Tarun Iype, Sasa Stefanic, Paola Picotti, Volodymyr M Korkhov</div> <em>eLife</em> (2022-08-18) <a href="https://doi.org/gsx57q">https://doi.org/gsx57q</a> <div class="csl-block">DOI: <a href="https://doi.org/10.7554/elife.77032">10.7554/elife.77032</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/35980026">35980026</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9433096">PMC9433096</a></div></div>
</div>
<div id="ref-UETOKcpN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">11. </div><div class="csl-right-inline"><strong>Structural basis of calmodulin modulation of the rod cyclic nucleotide-gated channel</strong> <div class="csl-block">Diane C A Barret, Dina Schuster, Matthew J Rodrigues, Alexander Leitner, Paola Picotti, Gebhard F X Schertler, U Benjamin Kaupp, Volodymyr M Korkhov, Jacopo Marino</div> <em>Proceedings of the National Academy of Sciences</em> (2023-04-03) <a href="https://doi.org/gs27gg">https://doi.org/gs27gg</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1073/pnas.2300309120">10.1073/pnas.2300309120</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/37011209">37011209</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10104587">PMC10104587</a></div></div>
</div>
<div id="ref-cYi0OSgq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">12. </div><div class="csl-right-inline"><strong>Regulatory sites of CaM-sensitive adenylyl cyclase AC8 revealed by cryo-EM and structural proteomics</strong> <div class="csl-block">Basavraj Khanppnavar, Dina Schuster, Pia Lavriha, Federico Uliana, Merve Özel, Ved Mehta, Alexander Leitner, Paola Picotti, Volodymyr M Korkhov</div> <em>EMBO Reports</em> (2024-02-13) <a href="https://doi.org/g9kwfz">https://doi.org/g9kwfz</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s44319-024-00076-y">10.1038/s44319-024-00076-y</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/38351373">38351373</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10933263">PMC10933263</a></div></div>
</div>
<div id="ref-HhtMYgJS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">13. </div><div class="csl-right-inline"><strong>A peptide-centric local stability assay enables proteome-scale identification of the protein targets and binding regions of diverse ligands</strong> <div class="csl-block">Kejia Li, Shijie Chen, Keyun Wang, Yan Wang, Lianji Xue, Yuying Ye, Zheng Fang, Jiawen Lyu, Haiyang Zhu, Yanan Li, … Mingliang Ye</div> <em>Nature Methods</em> (2024-12-10) <a href="https://doi.org/g84jvq">https://doi.org/g84jvq</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41592-024-02553-7">10.1038/s41592-024-02553-7</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/39658593">39658593</a></div></div>
</div>
<div id="ref-12oF7PyjF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">14. </div><div class="csl-right-inline"><strong>In‐Cell Fast Photochemical Oxidation Interrogates the Native Structure of Integral Membrane Proteins</strong> <div class="csl-block">Jie Sun, Mierxiati Saimi, Don Rempel, Qing Cao, Mengqi Chai, Weikai Li, Michael L Gross</div> <em>Angewandte Chemie International Edition</em> (2025-03-09) <a href="https://doi.org/g9kwfm">https://doi.org/g9kwfm</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/anie.202424779">10.1002/anie.202424779</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/40033852">40033852</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC12052488">PMC12052488</a></div></div>
</div>
<div id="ref-12LdlQcg6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">15. </div><div class="csl-right-inline"><strong>Thermal proteome profiling identifies the membrane-bound purinergic receptor P2X4 as a target of the autophagy inhibitor indophagolin</strong> <div class="csl-block">Marjorie A Carnero Corrales, Sarah Zinken, Georgios Konstantinidis, Muhammad Rafehi, Aliaa Abdelrahman, Yao-Wen Wu, Petra Janning, Christa E Müller, Luca Laraia, Herbert Waldmann</div> <em>Cell Chemical Biology</em> (2021-12) <a href="https://doi.org/gmz72w">https://doi.org/gmz72w</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.chembiol.2021.02.017">10.1016/j.chembiol.2021.02.017</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33725479">33725479</a></div></div>
</div>
<div id="ref-KSJWpyaf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">16. </div><div class="csl-right-inline"><strong>Cross-Linking Mass Spectrometry to Capture Protein Network Dynamics of Cell Membranome</strong> <div class="csl-block">Lucia Santorelli, Michele Costanzo, Sara Petrosino, Michele Santoro, Marianna Caterino, Margherita Ruoppolo, Paolo Grumati</div> <em>Methods in Molecular Biology</em> (2024-12-24) <a href="https://doi.org/g9kwfn">https://doi.org/g9kwfn</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/978-1-0716-4298-6_16">10.1007/978-1-0716-4298-6_16</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/39716008">39716008</a></div></div>
</div>
<div id="ref-ixF12fLE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">17. </div><div class="csl-right-inline"><strong>Cross-link assisted spatial proteomics to map sub-organelle proteomes and membrane protein topologies</strong> <div class="csl-block">Ying Zhu, Kerem Can Akkaya, Julia Ruta, Nanako Yokoyama, Cong Wang, Max Ruwolt, Diogo Borges Lima, Martin Lehmann, Fan Liu</div> <em>Nature Communications</em> (2024-04-17) <a href="https://doi.org/g9kwfv">https://doi.org/g9kwfv</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41467-024-47569-x">10.1038/s41467-024-47569-x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/38632225">38632225</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11024108">PMC11024108</a></div></div>
</div>
<div id="ref-1083OUMws" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">18. </div><div class="csl-right-inline"><strong>Glycan–protein cross-linking mass spectrometry reveals sialic acid-mediated protein networks on cell surfaces</strong> <div class="csl-block">Yixuan Xie, Siyu Chen, Qiongyu Li, Ying Sheng, Michael Russelle Alvarez, Joeriggo Reyes, Gege Xu, Kemal Solakyildirim, Carlito B Lebrilla</div> <em>Chemical Science</em> (2021) <a href="https://doi.org/g9kwf3">https://doi.org/g9kwf3</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1039/d1sc00814e">10.1039/d1sc00814e</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34257876">34257876</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8246274">PMC8246274</a></div></div>
</div>
<div id="ref-1DXKD97TB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">19. </div><div class="csl-right-inline"><strong>In situ cell-type-specific cell-surface proteomic profiling in mice</strong> <div class="csl-block">SAndrew Shuster, Jiefu Li, URee Chon, Miley C Sinantha-Hu, David J Luginbuhl, Namrata D Udeshi, Dominique Kiki Carey, Yukari H Takeo, Qijing Xie, Chuanyun Xu, … Liqun Luo</div> <em>Neuron</em> (2022-12) <a href="https://doi.org/gq3hq5">https://doi.org/gq3hq5</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.neuron.2022.09.025">10.1016/j.neuron.2022.09.025</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/36220098">36220098</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9742329">PMC9742329</a></div></div>
</div>
<div id="ref-1A0ELIyuD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">20. </div><div class="csl-right-inline"><strong>Profiling the proximal proteome of the activated μ-opioid receptor</strong> <div class="csl-block">Benjamin J Polacco, Braden T Lobingier, Emily E Blythe, Nohely Abreu, Prachi Khare, Matthew K Howard, Alberto J Gonzalez-Hernandez, Jiewei Xu, Qiongyu Li, Brandon Novy, … Ruth Hüttenhain</div> <em>Nature Chemical Biology</em> (2024-03-25) <a href="https://doi.org/gtqbbp">https://doi.org/gtqbbp</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41589-024-01588-3">10.1038/s41589-024-01588-3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/38528119">38528119</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11365811">PMC11365811</a></div></div>
</div>
<div id="ref-uJCnPykT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">21. </div><div class="csl-right-inline"><strong>Chromatographic Phospholipid Trapping for Automated H/D Exchange Mass Spectrometry of Membrane Protein–Lipid Assemblies</strong> <div class="csl-block">Dietmar Hammerschmid, Valeria Calvaresi, Chloe Bailey, Benjamin Russell Lewis, Argyris Politis, Michael Morris, Laetitia Denbigh, Malcolm Anderson, Eamonn Reading</div> <em>Analytical Chemistry</em> (2023-01-27) <a href="https://doi.org/g9kwms">https://doi.org/g9kwms</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.2c04876">10.1021/acs.analchem.2c04876</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/36706021">36706021</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9909672">PMC9909672</a></div></div>
</div>
<div id="ref-5zsiKMpi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">22. </div><div class="csl-right-inline"><strong>Molecular characterization of a complex of apoptosis-inducing factor 1 with cytochrome c oxidase of the mitochondrial respiratory chain</strong> <div class="csl-block">Johannes F Hevler, Riccardo Zenezeni Chiozzi, Alfredo Cabrera-Orefice, Ulrich Brandt, Susanne Arnold, Albert JR Heck</div> <em>Proceedings of the National Academy of Sciences</em> (2021-09-21) <a href="https://doi.org/g9kwf5">https://doi.org/g9kwf5</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1073/pnas.2106950118">10.1073/pnas.2106950118</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34548399">34548399</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8488679">PMC8488679</a></div></div>
</div>
<div id="ref-wqOAIazw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">23. </div><div class="csl-right-inline"><strong>A proximity proteomics pipeline with improved reproducibility and throughput</strong> <div class="csl-block">Xiaofang Zhong, Qiongyu Li, Benjamin J Polacco, Trupti Patil, Aaron Marley, Helene Foussard, Prachi Khare, Rasika Vartak, Jiewei Xu, Jeffrey F DiBerto, … Ruth Hüttenhain</div> <em>Molecular Systems Biology</em> (2024-07-01) <a href="https://doi.org/g9kwf2">https://doi.org/g9kwf2</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s44320-024-00049-2">10.1038/s44320-024-00049-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/38951684">38951684</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11297269">PMC11297269</a></div></div>
</div>
<div id="ref-QM5L157g" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">24. </div><div class="csl-right-inline"><strong>Capturing a rhodopsin receptor signalling cascade across a native membrane</strong> <div class="csl-block">Siyun Chen, Tamar Getter, David Salom, Di Wu, Daniel Quetschlich, Dror S Chorev, Krzysztof Palczewski, Carol V Robinson</div> <em>Nature</em> (2022-04-06) <a href="https://doi.org/g9kwfw">https://doi.org/g9kwfw</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41586-022-04547-x">10.1038/s41586-022-04547-x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/35388214">35388214</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9007743">PMC9007743</a></div></div>
</div>
<div id="ref-ehunITJ2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">25. </div><div class="csl-right-inline"><strong>Native Mass Spectrometry of Membrane Protein–Lipid Interactions in Different Detergent Environments</strong> <div class="csl-block">Smriti Kumar, Lauren Stover, Lie Wang, Hanieh Bahramimoghaddam, Ming Zhou, David H Russell, Arthur Laganowsky</div> <em>Analytical Chemistry</em> (2024-10-12) <a href="https://doi.org/g9kwfr">https://doi.org/g9kwfr</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.4c03312">10.1021/acs.analchem.4c03312</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/39394983">39394983</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11503522">PMC11503522</a></div></div>
</div>
<div id="ref-1AiSIssJj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">26. </div><div class="csl-right-inline"><strong>Native mass spectrometry of proteoliposomes containing integral and peripheral membrane proteins</strong> <div class="csl-block">Yun Zhu, Sangho D Yun, Tianqi Zhang, Jing-Yuan Chang, Lauren Stover, Arthur Laganowsky</div> <em>Chemical Science</em> (2023) <a href="https://doi.org/g9kwf4">https://doi.org/g9kwf4</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1039/d3sc04938h">10.1039/d3sc04938h</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/38098719">38098719</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10718073">PMC10718073</a></div></div>
</div>
<div id="ref-17kjm5RCK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">27. </div><div class="csl-right-inline"><strong>Capillary Zone Electrophoresis-Mass Spectrometry of Intact G Protein-Coupled Receptors Enables Proteoform Profiling</strong> <div class="csl-block">Ashley N Ives, Kevin Jooß, Rafael D Melani, Ryan T Fellers, John Janetzko, Neil L Kelleher</div> <em>Analytical Chemistry</em> (2025-03-27) <a href="https://doi.org/g9kwfs">https://doi.org/g9kwfs</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.4c06994">10.1021/acs.analchem.4c06994</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/40146057">40146057</a></div></div>
</div>
<div id="ref-1mGBvUhI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">28. </div><div class="csl-right-inline"><strong>Capillary Zone Electrophoresis-Tandem Mass Spectrometry for Top-Down Proteomics of Mouse Brain Integral Membrane Proteins</strong> <div class="csl-block">Qianjie Wang, Tian Xu, Fei Fang, Qianyi Wang, Peter Lundquist, Liangliang Sun</div> <em>Analytical Chemistry</em> (2023-08-18) <a href="https://doi.org/g9kwfq">https://doi.org/g9kwfq</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.3c02346">10.1021/acs.analchem.3c02346</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/37595263">37595263</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10540247">PMC10540247</a></div></div>
</div>
<div id="ref-19wOhPBGo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">29. </div><div class="csl-right-inline"><strong>The Effects of Sodium Ions on Ligand Binding and Conformational States of G Protein-Coupled Receptors—Insights from Mass Spectrometry</strong> <div class="csl-block">Mark T Agasid, Lars Sørensen, Leonhard H Urner, Jun Yan, Carol V Robinson</div> <em>Journal of the American Chemical Society</em> (2021-03-12) <a href="https://doi.org/gjf7x5">https://doi.org/gjf7x5</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/jacs.0c11837">10.1021/jacs.0c11837</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33711230">33711230</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7995251">PMC7995251</a></div></div>
</div>
<div id="ref-my3tj08K" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">30. </div><div class="csl-right-inline"><strong>Evaluation of Drug Responses to Human β&lt;sub&gt;2&lt;/sub&gt;AR Using Native Mass Spectrometry</strong> <div class="csl-block">Michiko Tajiri, Shunsuke Imai, Tsuyoshi Konuma, Keiko Shimamoto, Ichio Shimada, Satoko Akashi</div> <em>ACS Omega</em> (2023-06-28) <a href="https://doi.org/g9kwft">https://doi.org/g9kwft</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acsomega.3c02737">10.1021/acsomega.3c02737</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/37457453">37457453</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10339329">PMC10339329</a></div></div>
</div>
<div id="ref-2W5sjAvY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">31. </div><div class="csl-right-inline"><strong>Native mass spectrometry and structural studies reveal modulation of MsbA–nucleotide interactions by lipids</strong> <div class="csl-block">Tianqi Zhang, Jixing Lyu, Bowei Yang, Sangho D Yun, Elena Scott, Minglei Zhao, Arthur Laganowsky</div> <em>Nature Communications</em> (2024-07-15) <a href="https://doi.org/gt4mfb">https://doi.org/gt4mfb</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41467-024-50350-9">10.1038/s41467-024-50350-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/39009687">39009687</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11251056">PMC11251056</a></div></div>
</div>
<div id="ref-1AcwgRB7T" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">32. </div><div class="csl-right-inline"><strong>Mass spectrometry captures biased signalling and allosteric modulation of a G-protein-coupled receptor</strong> <div class="csl-block">Hsin-Yung Yen, Idlir Liko, Wanling Song, Parth Kapoor, Fernando Almeida, Joanna Toporowska, Karolina Gherbi, Jonathan TS Hopper, Steven J Charlton, Argyris Politis, … Carol V Robinson</div> <em>Nature Chemistry</em> (2022-11-10) <a href="https://doi.org/jmm7">https://doi.org/jmm7</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41557-022-01041-9">10.1038/s41557-022-01041-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/36357787">36357787</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9758051">PMC9758051</a></div></div>
</div>
<div id="ref-dNXuiXTr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">33. </div><div class="csl-right-inline"><strong>Native mass spectrometry prescreening of G protein-coupled receptor complexes for cryo-EM structure determination</strong> <div class="csl-block">Donggyun Kim, Weijing Liu, Rosa Viner, Vadim Cherezov</div> <em>Structure</em> (2024-12) <a href="https://doi.org/g9kwfp">https://doi.org/g9kwfp</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.str.2024.10.004">10.1016/j.str.2024.10.004</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/39471802">39471802</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11625002">PMC11625002</a></div></div>
</div>
<div id="ref-jPUY8z0i" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">34. </div><div class="csl-right-inline"><strong>Lipid- and protein-directed photosensitizer proximity labeling captures the cholesterol interactome</strong> <div class="csl-block">Andrew P Becker, Elijah Biletch, John Paul Kennelly, Ashley R Julio, Miranda Villaneuva, Rohith T Nagari, Daniel W Turner, Nikolas R Burton, Tomoyuki Fukuta, Liujuan Cui, … Keriann M Backus</div> <em>Cold Spring Harbor Laboratory</em> (2024-08-20) <a href="https://doi.org/g9kwhm">https://doi.org/g9kwhm</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/2024.08.20.608660">10.1101/2024.08.20.608660</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/39229057">39229057</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11370482">PMC11370482</a></div></div>
</div>
<div id="ref-KwvCLBXC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">35. </div><div class="csl-right-inline"><strong>A chemical proteomics approach for global mapping of functional lysines on cell surface of living cell</strong> <div class="csl-block">Ting Wang, Shiyun Ma, Guanghui Ji, Guoli Wang, Yang Liu, Lei Zhang, Ying Zhang, Haojie Lu</div> <em>Nature Communications</em> (2024-04-08) <a href="https://doi.org/g9kwhj">https://doi.org/g9kwhj</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41467-024-47033-w">10.1038/s41467-024-47033-w</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/38589397">38589397</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11001985">PMC11001985</a></div></div>
</div>
<div id="ref-jNq0LZ7n" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">36. </div><div class="csl-right-inline"><strong>Probing Monotopic Phosphoglycosyl Transferases from Complex Cellular Milieu</strong> <div class="csl-block">Alyssa J Anderson, Leah M Seebald, Christine A Arbour, Barbara Imperiali</div> <em>ACS Chemical Biology</em> (2022-11-08) <a href="https://doi.org/g6g3mb">https://doi.org/g6g3mb</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acschembio.2c00648">10.1021/acschembio.2c00648</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/36346917">36346917</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9703085">PMC9703085</a></div></div>
</div>
<div id="ref-8tpRy8vW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">37. </div><div class="csl-right-inline"><strong>Dual-Probe Activity-Based Protein Profiling Reveals Site-Specific Differences in Protein Binding of EGFR-Directed Drugs</strong> <div class="csl-block">Wouter van Bergen, Kristina Žuna, Jan Fiala, Elena E Pohl, Albert JR Heck, Marc P Baggelaar</div> <em>ACS Chemical Biology</em> (2024-07-25) <a href="https://doi.org/g9kwhh">https://doi.org/g9kwhh</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acschembio.3c00637">10.1021/acschembio.3c00637</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/39052621">39052621</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11334109">PMC11334109</a></div></div>
</div>
<div id="ref-5qPERR5E" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">38. </div><div class="csl-right-inline"><strong>Defining the Cell Surface Cysteinome Using Two-Step Enrichment Proteomics</strong> <div class="csl-block">Tianyang Yan, Lisa M Boatner, Liujuan Cui, Peter J Tontonoz, Keriann M Backus</div> <em>JACS Au</em> (2023-12-13) <a href="https://doi.org/g5c55k">https://doi.org/g5c55k</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/jacsau.3c00707">10.1021/jacsau.3c00707</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/38155636">38155636</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10751780">PMC10751780</a></div></div>
</div>
<div id="ref-ClHzzrGn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">39. </div><div class="csl-right-inline"><strong>A proteome-wide map of 20(S)-hydroxycholesterol interactors in cell membranes</strong> <div class="csl-block">Yu-Shiuan Cheng, Tianyi Zhang, Xiang Ma, Sarida Pratuangtham, Grace C Zhang, Alexander A Ondrus, Amirhossein Mafi, Brett Lomenick, Jeffrey J Jones, Alison E Ondrus</div> <em>Nature Chemical Biology</em> (2021-11-19) <a href="https://doi.org/g9kwhk">https://doi.org/g9kwhk</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41589-021-00907-2">10.1038/s41589-021-00907-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34799735">34799735</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8607797">PMC8607797</a></div></div>
</div>
<div id="ref-laLpQc2u" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">40. </div><div class="csl-right-inline"><strong>Cell Surface Engineering Enables Surfaceome Profiling</strong> <div class="csl-block">Zak Vilen, Abigail E Reeves, Timothy R O’Leary, Eugene Joeh, Naomi Kamasawa, Mia L Huang</div> <em>ACS Chemical Biology</em> (2022-04-20) <a href="https://doi.org/g9kwmw">https://doi.org/g9kwmw</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acschembio.1c00865">10.1021/acschembio.1c00865</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/35443134">35443134</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9901301">PMC9901301</a></div></div>
</div>
<div id="ref-HnAihA6J" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">41. </div><div class="csl-right-inline"><strong>Quantitative Glycan-Protein Cross-Linking Mass Spectrometry Using Enrichable Linkers Reveals Extensive Glycan-Mediated Protein Interaction Networks</strong> <div class="csl-block">Siyu Chen, Yixuan Xie, Michael Russelle Alvarez, Ying Sheng, Yasmine Bouchibti, Vincent Chang, Carlito B Lebrilla</div> <em>Analytical Chemistry</em> (2025-01-13) <a href="https://doi.org/g9kwmv">https://doi.org/g9kwmv</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.4c04134">10.1021/acs.analchem.4c04134</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/39805041">39805041</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11780575">PMC11780575</a></div></div>
</div>
<div id="ref-keHzgRUT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">42. </div><div class="csl-right-inline"><strong>Structural Mass Spectrometry Captures Residue-Resolved Comprehensive Conformational Rearrangements of a G Protein-Coupled Receptor</strong> <div class="csl-block">Hongyue Liu, Pengfei Yan, Zhaoyu Zhang, Hongbo Han, Qingtong Zhou, Jie Zheng, Jian Zhang, Fei Xu, Wenqing Shui</div> <em>Journal of the American Chemical Society</em> (2024-07-13) <a href="https://doi.org/g9kwmx">https://doi.org/g9kwmx</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/jacs.4c03922">10.1021/jacs.4c03922</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/39001877">39001877</a></div></div>
</div>
<div id="ref-eXw91M0D" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">43. </div><div class="csl-right-inline"><strong>Cryo-EM of soft-landed β-galactosidase: Gas-phase and native structures are remarkably similar</strong> <div class="csl-block">Tim K Esser, Jan Böhning, Alpcan Önür, Dinesh K Chinthapalli, Lukas Eriksson, Marko Grabarics, Paul Fremdling, Albert Konijnenberg, Alexander Makarov, Aurelien Botman, … Stephan Rauschenbach</div> <em>Science Advances</em> (2024-02-16) <a href="https://doi.org/gth2cp">https://doi.org/gth2cp</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1126/sciadv.adl4628">10.1126/sciadv.adl4628</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/38354247">38354247</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10866560">PMC10866560</a></div></div>
</div>
<div id="ref-OuTfCeL4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">44. </div><div class="csl-right-inline"><strong>Cryogenic Soft Landing Improves Structural Preservation of Protein Complexes</strong> <div class="csl-block">Michael S Westphall, Kenneth W Lee, Colin Hemme, Austin Z Salome, Keaton Mertz, Timothy Grant, Joshua J Coon</div> <em>Analytical Chemistry</em> (2023-09-21) <a href="https://doi.org/g9kwmt">https://doi.org/g9kwmt</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.3c03228">10.1021/acs.analchem.3c03228</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/37732836">37732836</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10568529">PMC10568529</a></div></div>
</div>
<div id="ref-Mak2jJSG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">45. </div><div class="csl-right-inline"><strong>Generating cysteine-trypsin cleavage sites with 2-chloroacetamidine capping</strong> <div class="csl-block">Samuel Ofori, Heta S Desai, Flowreen Shikwana, Lisa M Boatner, Emil R Dominguez III, José O Castellón, Keriann M Backus</div> <em>Chemical Communications</em> (2024) <a href="https://doi.org/g9kwmz">https://doi.org/g9kwmz</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1039/d4cc01583e">10.1039/d4cc01583e</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/39081146">39081146</a></div></div>
</div>
</div>
<!-- default theme -->

<style>
  /* import google fonts */
  @import url("https://fonts.googleapis.com/css?family=Open+Sans:400,600,700");
  @import url("https://fonts.googleapis.com/css?family=Source+Code+Pro");

  /* -------------------------------------------------- */
  /* global */
  /* -------------------------------------------------- */

  /* all elements */
  * {
    /* force sans-serif font unless specified otherwise */
    font-family: "Open Sans", "Helvetica", sans-serif;

    /* prevent text inflation on some mobile browsers */
    -webkit-text-size-adjust: none !important;
    -moz-text-size-adjust: none !important;
    -o-text-size-adjust: none !important;
    text-size-adjust: none !important;
  }

  @media only screen {
    /* "page" element */
    body {
      position: relative;
      box-sizing: border-box;
      font-size: 12pt;
      line-height: 1.5;
      max-width: 8.5in;
      margin: 20px auto;
      padding: 40px;
      border-radius: 5px;
      border: solid 1px #bdbdbd;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      background: #ffffff;
    }
  }

  /* when on screen < 8.5in wide */
  @media only screen and (max-width: 8.5in) {
    /* "page" element */
    body {
      padding: 20px;
      margin: 0;
      border-radius: 0;
      border: none;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05) inset;
      background: none;
    }
  }

  /* -------------------------------------------------- */
  /* headings */
  /* -------------------------------------------------- */

  /* all headings */
  h1,
  h2,
  h3,
  h4,
  h5,
  h6 {
    margin: 20px 0;
    padding: 0;
    font-weight: bold;
  }

  /* biggest heading */
  h1 {
    margin: 40px 0;
    text-align: center;
  }

  /* second biggest heading */
  h2 {
    margin-top: 30px;
    padding-bottom: 5px;
    border-bottom: solid 1px #bdbdbd;
  }

  /* heading font sizes */
  h1 {
    font-size: 2em;
  }
  h2 {
    font-size: 1.5em;
  }
  h3 {
    font-size: 1.35em;
  }
  h4 {
    font-size: 1.25em;
  }
  h5 {
    font-size: 1.15em;
  }
  h6 {
    font-size: 1em;
  }

  /* -------------------------------------------------- */
  /* manuscript header */
  /* -------------------------------------------------- */

  /* manuscript title */
  header > h1 {
    margin: 0;
  }

  /* manuscript title caption text (ie "automatically generated on") */
  header + p {
    text-align: center;
    margin-top: 10px;
  }

  /* -------------------------------------------------- */
  /* text elements */
  /* -------------------------------------------------- */

  /* links */
  a {
    color: #2196f3;
    overflow-wrap: break-word;
  }

  /* superscripts and subscripts */
  sub,
  sup {
    /* prevent from affecting line height */
    line-height: 0;
  }

  /* unordered and ordered lists*/
  ul,
  ol {
    padding-left: 20px;
  }

  /* class for styling text semibold */
  .semibold {
    font-weight: 600;
  }

  /* class for styling elements horizontally left aligned */
  .left {
    display: block;
    text-align: left;
    margin-left: auto;
    margin-right: 0;
    justify-content: left;
  }

  /* class for styling elements horizontally centered */
  .center {
    display: block;
    text-align: center;
    margin-left: auto;
    margin-right: auto;
    justify-content: center;
  }

  /* class for styling elements horizontally right aligned */
  .right {
    display: block;
    text-align: right;
    margin-left: 0;
    margin-right: auto;
    justify-content: right;
  }

  /* -------------------------------------------------- */
  /* section elements */
  /* -------------------------------------------------- */

  /* horizontal divider line */
  hr {
    border: none;
    height: 1px;
    background: #bdbdbd;
  }

  /* paragraphs, horizontal dividers, figures, tables, code */
  p,
  hr,
  figure,
  table,
  pre {
    /* treat all as "paragraphs", with consistent vertical margins */
    margin-top: 20px;
    margin-bottom: 20px;
  }

  /* -------------------------------------------------- */
  /* figures */
  /* -------------------------------------------------- */

  /* figure */
  figure {
    max-width: 100%;
    margin-left: auto;
    margin-right: auto;
  }

  /* figure caption */
  figcaption {
    padding: 0;
    padding-top: 10px;
  }

  /* figure image element */
  figure > img,
  figure > svg {
    max-width: 100%;
    display: block;
    margin-left: auto;
    margin-right: auto;
  }

  /* figure auto-number */
  img + figcaption > span:first-of-type,
  svg + figcaption > span:first-of-type {
    font-weight: bold;
    margin-right: 5px;
  }

  /* -------------------------------------------------- */
  /* tables */
  /* -------------------------------------------------- */

  /* table */
  table {
    border-collapse: collapse;
    border-spacing: 0;
    width: 100%;
    margin-left: auto;
    margin-right: auto;
  }

  /* table cells */
  th,
  td {
    border: solid 1px #bdbdbd;
    padding: 10px;
    /* squash table if too wide for page by forcing line breaks */
    overflow-wrap: break-word;
    word-break: break-word;
  }

  /* header row and even rows */
  th,
  tr:nth-child(2n) {
    background-color: #fafafa;
  }

  /* odd rows */
  tr:nth-child(2n + 1) {
    background-color: #ffffff;
  }

  /* table caption */
  caption {
    text-align: left;
    padding: 0;
    padding-bottom: 10px;
  }

  /* table auto-number */
  table > caption > span:first-of-type {
    font-weight: bold;
    margin-right: 5px;
  }

  /* -------------------------------------------------- */
  /* code */
  /* -------------------------------------------------- */

  /* multi-line code block */
  pre {
    padding: 10px;
    background-color: #eeeeee;
    color: #000000;
    border-radius: 5px;
    break-inside: avoid;
    text-align: left;
  }

  /* inline code, ie code within normal text */
  :not(pre) > code {
    padding: 0 4px;
    background-color: #eeeeee;
    color: #000000;
    border-radius: 5px;
  }

  /* code text */
  /* apply all children, to reach syntax highlighting sub-elements */
  code,
  code * {
    /* force monospace font */
    font-family: "Source Code Pro", "Courier New", monospace;
  }

  /* -------------------------------------------------- */
  /* quotes */
  /* -------------------------------------------------- */

  /* quoted text */
  blockquote {
    margin: 0;
    padding: 0;
    border-left: 4px solid #bdbdbd;
    padding-left: 16px;
    break-inside: avoid;
  }

  /* -------------------------------------------------- */
  /* banners */
  /* -------------------------------------------------- */

  /* info banners */
  .banner {
    box-sizing: border-box;
    display: block;
    position: relative;
    width: 100%;
    margin-top: 20px;
    margin-bottom: 20px;
    padding: 20px;
    text-align: center;
  }

  /* paragraph in banner */
  .banner > p {
    margin: 0;
  }

  /* -------------------------------------------------- */
  /* highlight colors */
  /* -------------------------------------------------- */

  .white {
    background: #ffffff;
  }
  .lightgrey {
    background: #eeeeee;
  }
  .grey {
    background: #757575;
  }
  .darkgrey {
    background: #424242;
  }
  .black {
    background: #000000;
  }
  .lightred {
    background: #ffcdd2;
  }
  .lightyellow {
    background: #ffecb3;
  }
  .lightgreen {
    background: #dcedc8;
  }
  .lightblue {
    background: #e3f2fd;
  }
  .lightpurple {
    background: #f3e5f5;
  }
  .red {
    background: #f44336;
  }
  .orange {
    background: #ff9800;
  }
  .yellow {
    background: #ffeb3b;
  }
  .green {
    background: #4caf50;
  }
  .blue {
    background: #2196f3;
  }
  .purple {
    background: #9c27b0;
  }
  .white,
  .lightgrey,
  .lightred,
  .lightyellow,
  .lightgreen,
  .lightblue,
  .lightpurple,
  .orange,
  .yellow,
  .white a,
  .lightgrey a,
  .lightred a,
  .lightyellow a,
  .lightgreen a,
  .lightblue a,
  .lightpurple a,
  .orange a,
  .yellow a {
    color: #000000;
  }
  .grey,
  .darkgrey,
  .black,
  .red,
  .green,
  .blue,
  .purple,
  .grey a,
  .darkgrey a,
  .black a,
  .red a,
  .green a,
  .blue a,
  .purple a {
    color: #ffffff;
  }

  /* -------------------------------------------------- */
  /* buttons */
  /* -------------------------------------------------- */

  /* class for styling links like buttons */
  .button {
    display: inline-flex;
    justify-content: center;
    align-items: center;
    margin: 5px;
    padding: 10px 20px;
    font-size: 0.75em;
    font-weight: 600;
    text-transform: uppercase;
    text-decoration: none;
    letter-spacing: 1px;
    background: none;
    color: #2196f3;
    border: solid 1px #bdbdbd;
    border-radius: 5px;
  }

  /* buttons when hovered */
  .button:hover:not([disabled]),
  .icon_button:hover:not([disabled]) {
    cursor: pointer;
    background: #f5f5f5;
  }

  /* buttons when disabled */
  .button[disabled],
  .icon_button[disabled] {
    opacity: 0.35;
    pointer-events: none;
  }

  /* class for styling buttons containg only single icon */
  .icon_button {
    display: inline-flex;
    justify-content: center;
    align-items: center;
    text-decoration: none;
    margin: 0;
    padding: 0;
    background: none;
    border-radius: 5px;
    border: none;
    width: 20px;
    height: 20px;
    min-width: 20px;
    min-height: 20px;
  }

  /* icon button inner svg image */
  .icon_button > svg {
    height: 16px;
  }

  /* -------------------------------------------------- */
  /* icons */
  /* -------------------------------------------------- */

  /* class for styling icons inline with text */
  .inline_icon {
    height: 1em;
    position: relative;
    top: 0.125em;
  }

  /* -------------------------------------------------- */
  /* references */
  /* -------------------------------------------------- */

  .csl-entry {
    margin-top: 15px;
    margin-bottom: 15px;
  }

  /* -------------------------------------------------- */
  /* print control */
  /* -------------------------------------------------- */

  @media print {
    @page {
      /* suggested printing margin */
      margin: 0.5in;
    }

    /* document and "page" elements */
    html,
    body {
      margin: 0;
      padding: 0;
      width: 100%;
      height: 100%;
    }

    /* "page" element */
    body {
      font-size: 11pt !important;
      line-height: 1.35;
    }

    /* all headings */
    h1,
    h2,
    h3,
    h4,
    h5,
    h6 {
      margin: 15px 0;
    }

    /* figures and tables */
    figure,
    table {
      font-size: 0.85em;
    }

    /* table cells */
    th,
    td {
      padding: 5px;
    }

    /* shrink font awesome icons */
    i.fas,
    i.fab,
    i.far,
    i.fal {
      transform: scale(0.85);
    }

    /* decrease banner margins */
    .banner {
      margin-top: 15px;
      margin-bottom: 15px;
      padding: 15px;
    }

    /* class for centering an element vertically on its own page */
    .page_center {
      margin: auto;
      width: 100%;
      height: 100%;
      display: flex;
      align-items: center;
      vertical-align: middle;
      break-before: page;
      break-after: page;
    }

    /* always insert a page break before the element */
    .page_break_before {
      break-before: page;
    }

    /* always insert a page break after the element */
    .page_break_after {
      break-after: page;
    }

    /* avoid page break before the element */
    .page_break_before_avoid {
      break-before: avoid;
    }

    /* avoid page break after the element */
    .page_break_after_avoid {
      break-after: avoid;
    }

    /* avoid page break inside the element */
    .page_break_inside_avoid {
      break-inside: avoid;
    }
  }

  /* -------------------------------------------------- */
  /* override pandoc css quirks */
  /* -------------------------------------------------- */

  .sourceCode {
    /* prevent unsightly overflow in wide code blocks */
    overflow: auto !important;
  }

  div.sourceCode {
    /* prevent background fill on top-most code block  container */
    background: none !important;
  }

  .sourceCode * {
    /* force consistent line spacing */
    line-height: 1.5 !important;
  }

  div.sourceCode {
    /* style code block margins same as <pre> element */
    margin-top: 20px;
    margin-bottom: 20px;
  }

  /* -------------------------------------------------- */
  /* tablenos */
  /* -------------------------------------------------- */

  /* tablenos wrapper */
  .tablenos {
    width: 100%;
    margin: 20px 0;
  }

  .tablenos > table {
    /* move margins from table to table_wrapper to allow margin collapsing */
    margin: 0;
  }

  @media only screen {
    /* tablenos wrapper */
    .tablenos {
      /* show scrollbar on tables if necessary to prevent overflow */
      overflow-x: auto !important;
    }

    .tablenos th,
    .tablenos td {
      overflow-wrap: unset !important;
      word-break: unset !important;
    }

    /* table in wrapper */
    .tablenos table,
    .tablenos table * {
      /* don't break table words */
      overflow-wrap: normal !important;
    }
  }
</style>
<!-- 
    Plugin Core

    Functions needed for and shared across all first-party plugins.
-->

<script>
  // get element that is target of hash (from link element or url)
  function getHashTarget(link) {
    const hash = link ? link.hash : window.location.hash;
    const id = hash.slice(1);
    let target = document.querySelector(`[id="${id}"]`);
    if (!target) return;

    // if figure or table, modify target to get expected element
    if (id.indexOf("fig:") === 0) target = target.querySelector("figure");
    if (id.indexOf("tbl:") === 0) target = target.querySelector("table");

    return target;
  }

  // get position/dimensions of element or viewport
  function getRectInView(element) {
    let rect = {};
    rect.left = 0;
    rect.top = 0;
    rect.right = document.documentElement.clientWidth;
    rect.bottom = document.documentElement.clientHeight;
    let style = {};

    if (element instanceof HTMLElement) {
      rect = element.getBoundingClientRect();
      style = window.getComputedStyle(element);
    }

    const margin = {};
    margin.left = parseFloat(style.marginLeftWidth) || 0;
    margin.top = parseFloat(style.marginTopWidth) || 0;
    margin.right = parseFloat(style.marginRightWidth) || 0;
    margin.bottom = parseFloat(style.marginBottomWidth) || 0;

    const border = {};
    border.left = parseFloat(style.borderLeftWidth) || 0;
    border.top = parseFloat(style.borderTopWidth) || 0;
    border.right = parseFloat(style.borderRightWidth) || 0;
    border.bottom = parseFloat(style.borderBottomWidth) || 0;

    const newRect = {};
    newRect.left = rect.left + margin.left + border.left;
    newRect.top = rect.top + margin.top + border.top;
    newRect.right = rect.right + margin.right + border.right;
    newRect.bottom = rect.bottom + margin.bottom + border.bottom;
    newRect.width = newRect.right - newRect.left;
    newRect.height = newRect.bottom - newRect.top;

    return newRect;
  }

  // get position of element relative to page
  function getRectInPage(element) {
    const rect = getRectInView(element);
    const body = getRectInView(document.body);

    const newRect = {};
    newRect.left = rect.left - body.left;
    newRect.top = rect.top - body.top;
    newRect.right = rect.right - body.left;
    newRect.bottom = rect.bottom - body.top;
    newRect.width = rect.width;
    newRect.height = rect.height;

    return newRect;
  }

  // get closest element before specified element that matches query
  function firstBefore(element, query) {
    while (element && element !== document.body && !element.matches(query))
      element = element.previousElementSibling || element.parentNode;

    return element;
  }

  // check if element is part of collapsed heading
  function isCollapsed(element) {
    while (element && element !== document.body) {
      if (element.dataset.collapsed === "true") return true;
      element = element.parentNode;
    }
    return false;
  }

  // expand any collapsed parent containers of element if necessary
  function expandElement(element) {
    if (isCollapsed(element)) {
      // accordion plugin
      const heading = firstBefore(element, "h2");
      if (heading) heading.click();
      // details/summary HTML element
      const summary = firstBefore(element, "summary");
      if (summary) summary.click();
    }
  }

  // scroll to and focus element
  function goToElement(element, offset) {
    // expand accordion section if collapsed
    expandElement(element);
    const y =
      getRectInView(element).top -
      getRectInView(document.documentElement).top -
      (offset || 0);

    // trigger any function listening for "onscroll" event
    window.dispatchEvent(new Event("scroll"));
    window.scrollTo(0, y);
    document.activeElement.blur();
    element.focus();
  }

  // get list of elements after a start element up to element matching query
  function nextUntil(element, query, exclude) {
    const elements = [];
    while (((element = element.nextElementSibling), element)) {
      if (element.matches(query)) break;
      if (!element.matches(exclude)) elements.push(element);
    }
    return elements;
  }
</script>
<!--
  Accordion Plugin

  Allows sections of content under h2 headings to be collapsible.
-->

<script type="module">
  // whether to always start expanded ('false'), always start collapsed
  // ('true'), or start collapsed when screen small ('auto')
  const startCollapsed = "auto";

  // start script
  function start() {
    // run through each <h2> heading
    const headings = document.querySelectorAll("h2");
    for (const heading of headings) {
      addArrow(heading);

      // start expanded/collapsed based on option
      if (
        startCollapsed === "true" ||
        (startCollapsed === "auto" && isSmallScreen()) ||
        heading.dataset.collapsed === "true"
      )
        collapseHeading(heading);
      else expandElement(heading);
    }

    // attach hash change listener to window
    window.addEventListener("hashchange", onHashChange);
  }

  // when hash (eg manuscript.html#introduction) changes
  function onHashChange() {
    const target = getHashTarget();
    if (target) goToElement(target);
  }

  // add arrow to heading
  function addArrow(heading) {
    // add arrow button
    const arrow = document.createElement("button");
    arrow.innerHTML = document.querySelector(".icon_angle_down").innerHTML;
    arrow.classList.add("icon_button", "accordion_arrow");
    heading.insertBefore(arrow, heading.firstChild);

    // attach click listener to heading and button
    heading.addEventListener("click", onHeadingClick);
    arrow.addEventListener("click", onArrowClick);
  }

  // determine if on mobile-like device with small screen
  function isSmallScreen() {
    return Math.min(window.innerWidth, window.innerHeight) < 480;
  }

  // when <h2> heading is clicked
  function onHeadingClick(event) {
    // only collapse if <h2> itself is target of click (eg, user did
    // not click on anchor within <h2>)
    if (event.target === this) toggleCollapse(this);
  }

  // when arrow button is clicked
  function onArrowClick() {
    toggleCollapse(this.parentNode);
  }

  // collapse section if expanded, expand if collapsed
  function toggleCollapse(heading) {
    if (heading.dataset.collapsed === "false") collapseHeading(heading);
    else expandElement(heading);
  }

  // elements to exclude from collapse, such as table of contents panel,
  // hypothesis panel, etc
  const exclude = "#toc_panel, div.annotator-frame, #lightbox_overlay";

  // collapse section
  function collapseHeading(heading) {
    heading.setAttribute("data-collapsed", "true");
    const children = getChildren(heading);
    for (const child of children) child.setAttribute("data-collapsed", "true");
  }

  // expand section
  function expandElement(heading) {
    heading.setAttribute("data-collapsed", "false");
    const children = getChildren(heading);
    for (const child of children) child.setAttribute("data-collapsed", "false");
  }

  // get list of elements between this <h2> and next <h2> or <h1>
  // ("children" of the <h2> section)
  function getChildren(heading) {
    return nextUntil(heading, "h2, h1", exclude);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- angle down icon -->

<template class="icon_angle_down">
  <!-- modified from: https://fontawesome.com/icons/angle-down -->
  <svg width="16" height="16" viewBox="0 0 448 512">
    <path
      fill="currentColor"
      d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* accordion arrow button */
    .accordion_arrow {
      margin-right: 10px;
    }

    /* arrow icon when <h2> data-collapsed attribute true */
    h2[data-collapsed="true"] > .accordion_arrow > svg {
      transform: rotate(-90deg);
    }

    /* all elements (except <h2>'s) when data-collapsed attribute true */
    *:not(h2)[data-collapsed="true"] {
      display: none;
    }

    /* accordion arrow button when hovered and <h2>'s when hovered */
    .accordion_arrow:hover,
    h2[data-collapsed="true"]:hover,
    h2[data-collapsed="false"]:hover {
      cursor: pointer;
    }
  }

  /* always hide accordion arrow button on print */
  @media only print {
    .accordion_arrow {
      display: none;
    }
  }
</style>
<!--
  Anchors Plugin

  Adds an anchor next to each of a certain type of element that provides a
  human-readable url to that specific item/position in the document (e.g.
  "manuscript.html#abstract"). It also makes it such that scrolling out of view
  of a target removes its identifier from the url.
-->

<script type="module">
  // which types of elements to add anchors next to, in "document.querySelector"
  // format
  const typesQuery =
    'h1, h2, h3, div[id^="fig:"], div[id^="tbl:"], span[id^="eq:"]';

  // start script
  function start() {
    // add anchor to each element of specified types
    const elements = document.querySelectorAll(typesQuery);
    for (const element of elements) addAnchor(element);

    // attach scroll listener to window
    window.addEventListener("scroll", onScroll);
  }

  // when window is scrolled
  function onScroll() {
    // if url has hash and user has scrolled out of view of hash
    // target, remove hash from url
    const tolerance = 100;
    const target = getHashTarget();
    if (target) {
      if (
        target.getBoundingClientRect().top > window.innerHeight + tolerance ||
        target.getBoundingClientRect().bottom < 0 - tolerance
      )
        history.pushState(null, null, " ");
    }
  }

  // add anchor to element
  function addAnchor(element) {
    let addTo; // element to add anchor button to

    // if figure or table, modify withId and addTo to get expected
    // elements
    if (element.id.indexOf("fig:") === 0) {
      addTo = element.querySelector("figcaption");
    } else if (element.id.indexOf("tbl:") === 0) {
      addTo = element.querySelector("caption");
    } else if (element.id.indexOf("eq:") === 0) {
      addTo = element.querySelector(".eqnos-number");
    }

    addTo = addTo || element;
    const id = element.id || null;

    // do not add anchor if element doesn't have assigned id.
    // id is generated by pandoc and is assumed to be unique and
    // human-readable
    if (!id) return;

    // create anchor button
    const anchor = document.createElement("a");
    anchor.innerHTML = document.querySelector(".icon_link").innerHTML;
    anchor.title = "Link to this part of the document";
    anchor.classList.add("icon_button", "anchor");
    anchor.dataset.ignore = "true";
    anchor.href = "#" + id;
    addTo.appendChild(anchor);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- link icon -->

<template class="icon_link">
  <!-- modified from: https://fontawesome.com/icons/link -->
  <svg width="16" height="16" viewBox="0 0 512 512">
    <path
      fill="currentColor"
      d="M326.612 185.391c59.747 59.809 58.927 155.698.36 214.59-.11.12-.24.25-.36.37l-67.2 67.2c-59.27 59.27-155.699 59.262-214.96 0-59.27-59.26-59.27-155.7 0-214.96l37.106-37.106c9.84-9.84 26.786-3.3 27.294 10.606.648 17.722 3.826 35.527 9.69 52.721 1.986 5.822.567 12.262-3.783 16.612l-13.087 13.087c-28.026 28.026-28.905 73.66-1.155 101.96 28.024 28.579 74.086 28.749 102.325.51l67.2-67.19c28.191-28.191 28.073-73.757 0-101.83-3.701-3.694-7.429-6.564-10.341-8.569a16.037 16.037 0 0 1-6.947-12.606c-.396-10.567 3.348-21.456 11.698-29.806l21.054-21.055c5.521-5.521 14.182-6.199 20.584-1.731a152.482 152.482 0 0 1 20.522 17.197zM467.547 44.449c-59.261-59.262-155.69-59.27-214.96 0l-67.2 67.2c-.12.12-.25.25-.36.37-58.566 58.892-59.387 154.781.36 214.59a152.454 152.454 0 0 0 20.521 17.196c6.402 4.468 15.064 3.789 20.584-1.731l21.054-21.055c8.35-8.35 12.094-19.239 11.698-29.806a16.037 16.037 0 0 0-6.947-12.606c-2.912-2.005-6.64-4.875-10.341-8.569-28.073-28.073-28.191-73.639 0-101.83l67.2-67.19c28.239-28.239 74.3-28.069 102.325.51 27.75 28.3 26.872 73.934-1.155 101.96l-13.087 13.087c-4.35 4.35-5.769 10.79-3.783 16.612 5.864 17.194 9.042 34.999 9.69 52.721.509 13.906 17.454 20.446 27.294 10.606l37.106-37.106c59.271-59.259 59.271-155.699.001-214.959z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* anchor button */
    .anchor {
      opacity: 0;
      margin-left: 5px;
    }

    /* anchor buttons within <h2>'s */
    h2 .anchor {
      margin-left: 10px;
    }

    /* anchor buttons when hovered/focused and anything containing an anchor button when hovered */
    *:hover > .anchor,
    .anchor:hover,
    .anchor:focus {
      opacity: 1;
    }

    /* anchor button when hovered */
    .anchor:hover {
      cursor: pointer;
    }
  }

  /* always show anchor button on devices with no mouse/hover ability */
  @media (hover: none) {
    .anchor {
      opacity: 1;
    }
  }

  /* always hide anchor button on print */
  @media only print {
    .anchor {
      display: none;
    }
  }
</style>
<!-- 
    Attributes Plugin

    Allows arbitrary HTML attributes to be attached to (almost) any element.
    Place an HTML comment inside or next to the desired element with the content:
    $attribute="value"
-->

<script type="module">
  // start script
  function start() {
    // get list of comments in document
    const comments = findComments();

    for (const comment of comments)
      if (comment.parentElement)
        addAttributes(comment.parentElement, comment.nodeValue.trim());
  }

  // add html attributes to specified element based on string of
  // html attributes and values
  function addAttributes(element, text) {
    // regex's for finding attribute/value pairs in the format of
    // attribute="value" or attribute='value
    const regex2 = /\$([a-zA-Z\-]+)?=\"(.+?)\"/;
    const regex1 = /\$([a-zA-Z\-]+)?=\'(.+?)\'/;

    // loop through attribute/value pairs
    let match;
    while ((match = text.match(regex2) || text.match(regex1))) {
      // get attribute and value from regex capture groups
      let attribute = match[1];
      let value = match[2];

      // remove from string
      text = text.substring(match.index + match[0].length);

      if (!attribute || !value) break;

      // set attribute of parent element
      try {
        element.setAttribute(attribute, value);
      } catch (error) {
        console.log(error);
      }

      // special case for colspan
      if (attribute === "colspan") removeTableCells(element, value);
    }
  }

  // get list of comment elements in document
  function findComments() {
    const comments = [];

    // iterate over comment nodes in document
    function acceptNode(node) {
      return NodeFilter.FILTER_ACCEPT;
    }
    const iterator = document.createNodeIterator(
      document.body,
      NodeFilter.SHOW_COMMENT,
      acceptNode
    );
    let node;
    while ((node = iterator.nextNode())) comments.push(node);

    return comments;
  }

  // remove certain number of cells after specified cell
  function removeTableCells(cell, number) {
    number = parseInt(number);
    if (!number) return;

    // remove elements
    for (; number > 1; number--) {
      if (cell.nextElementSibling) cell.nextElementSibling.remove();
    }
  }

  // start script on DOMContentLoaded instead of load to ensure this plugins
  // runs before other plugins
  window.addEventListener("DOMContentLoaded", start);
</script>
<!--
  Jump to First Plugin

  Adds a button next to each reference entry, figure, and table that jumps the
  page to the first occurrence of a link to that item in the manuscript.
-->

<script type="module">
  // whether to add buttons next to reference entries
  const references = "true";
  // whether to add buttons next to figures
  const figures = "true";
  // whether to add buttons next to tables
  const tables = "true";

  // start script
  function start() {
    if (references !== "false")
      makeButtons(`div[id^="ref-"]`, ".csl-left-margin", "reference");
    if (figures !== "false")
      makeButtons(`div[id^="fig:"]`, "figcaption", "figure");
    if (tables !== "false") makeButtons(`div[id^="tbl:"]`, "caption", "table");
  }

  // when jump button clicked
  function onButtonClick() {
    const first = getFirstOccurrence(this.dataset.id);
    if (!first) return;

    // update url hash so navigating "back" in history will return user to button
    window.location.hash = this.dataset.id;
    // scroll to link
    const timeout = function () {
      goToElement(first, window.innerHeight * 0.5);
    };
    window.setTimeout(timeout, 0);
  }

  // get first occurrence of link to item in document
  function getFirstOccurrence(id) {
    let query = "a";
    query += '[href="#' + id + '"]';
    // exclude buttons, anchor links, toc links, etc
    query += ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    return document.querySelector(query);
  }

  // add button next to each reference entry, figure, or table
  function makeButtons(query, containerQuery, subject) {
    const elements = document.querySelectorAll(query);
    for (const element of elements) {
      const id = element.id;
      const buttonContainer = element.querySelector(containerQuery);
      const first = getFirstOccurrence(id);

      // if can't find link to reference or place to put button, ignore
      if (!first || !buttonContainer) continue;

      // make jump button
      let button = document.createElement("button");
      button.classList.add("icon_button", "jump_arrow");
      button.title = `Jump to the first occurrence of this ${subject} in the document`;
      const icon = document.querySelector(".icon_angle_double_up");
      button.innerHTML = icon.innerHTML;
      button.dataset.id = id;
      button.dataset.ignore = "true";
      button.addEventListener("click", onButtonClick);
      buttonContainer.prepend(button);
    }
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- angle double up icon -->

<template class="icon_angle_double_up">
  <!-- modified from: https://fontawesome.com/icons/angle-double-up -->
  <svg width="16" height="16" viewBox="0 0 320 512">
    <path
      fill="currentColor"
      d="M177 255.7l136 136c9.4 9.4 9.4 24.6 0 33.9l-22.6 22.6c-9.4 9.4-24.6 9.4-33.9 0L160 351.9l-96.4 96.4c-9.4 9.4-24.6 9.4-33.9 0L7 425.7c-9.4-9.4-9.4-24.6 0-33.9l136-136c9.4-9.5 24.6-9.5 34-.1zm-34-192L7 199.7c-9.4 9.4-9.4 24.6 0 33.9l22.6 22.6c9.4 9.4 24.6 9.4 33.9 0l96.4-96.4 96.4 96.4c9.4 9.4 24.6 9.4 33.9 0l22.6-22.6c9.4-9.4 9.4-24.6 0-33.9l-136-136c-9.2-9.4-24.4-9.4-33.8 0z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* jump button */
    .jump_arrow {
      position: relative;
      top: 0.125em;
      margin-right: 5px;
    }
  }

  /* always hide jump button on print */
  @media only print {
    .jump_arrow {
      display: none;
    }
  }
</style>
<!-- 
    Lightbox Plugin

    Makes it such that when a user clicks on an image, the image fills the
    screen and the user can pan/drag/zoom the image and navigate between other
    images in the document.
-->

<script type="module">
  // list of possible zoom/scale factors
  const zooms =
    "0.1, 0.25, 0.333333, 0.5, 0.666666, 0.75, 1, 1.25, 1.5, 1.75, 2, 2.5, 3, 3.5, 4, 5, 6, 7, 8";
  // whether to fit image to view ('fit'), display at 100% and shrink if
  // necessary ('shrink'), or always display at 100% ('100')
  const defaultZoom = "fit";
  // whether to zoom in/out toward center of view ('true') or mouse ('false')
  const centerZoom = "false";

  // start script
  function start() {
    // run through each <img> element
    const imgs = document.querySelectorAll("figure > img");
    let count = 1;
    for (const img of imgs) {
      img.classList.add("lightbox_document_img");
      img.dataset.number = count;
      img.dataset.total = imgs.length;
      img.addEventListener("click", openLightbox);
      count++;
    }

    // attach mouse and key listeners to window
    window.addEventListener("mousemove", onWindowMouseMove);
    window.addEventListener("keyup", onKeyUp);
  }

  // when mouse is moved anywhere in window
  function onWindowMouseMove(event) {
    window.mouseX = event.clientX;
    window.mouseY = event.clientY;
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    switch (event.key) {
      // trigger click of prev button
      case "ArrowLeft":
        const prevButton = document.getElementById("lightbox_prev_button");
        if (prevButton) prevButton.click();
        break;
      // trigger click of next button
      case "ArrowRight":
        const nextButton = document.getElementById("lightbox_next_button");
        if (nextButton) nextButton.click();
        break;
      // close on esc
      case "Escape":
        closeLightbox();
        break;
    }
  }

  // open lightbox
  function openLightbox() {
    const lightbox = makeLightbox(this);
    if (!lightbox) return;

    blurBody(lightbox);
    document.body.appendChild(lightbox);
  }

  // make lightbox
  function makeLightbox(img) {
    // delete lightbox if it exists, start fresh
    closeLightbox();

    // create screen overlay containing lightbox
    const overlay = document.createElement("div");
    overlay.id = "lightbox_overlay";

    // create image info boxes
    const numberInfo = document.createElement("div");
    const zoomInfo = document.createElement("div");
    numberInfo.id = "lightbox_number_info";
    zoomInfo.id = "lightbox_zoom_info";

    // create container for image
    const imageContainer = document.createElement("div");
    imageContainer.id = "lightbox_image_container";
    const lightboxImg = makeLightboxImg(
      img,
      imageContainer,
      numberInfo,
      zoomInfo
    );
    imageContainer.appendChild(lightboxImg);

    // create bottom container for caption and navigation buttons
    const bottomContainer = document.createElement("div");
    bottomContainer.id = "lightbox_bottom_container";
    const caption = makeCaption(img);
    const prevButton = makePrevButton(img);
    const nextButton = makeNextButton(img);
    bottomContainer.appendChild(prevButton);
    bottomContainer.appendChild(caption);
    bottomContainer.appendChild(nextButton);

    // attach top middle and bottom to overlay
    overlay.appendChild(numberInfo);
    overlay.appendChild(zoomInfo);
    overlay.appendChild(imageContainer);
    overlay.appendChild(bottomContainer);

    return overlay;
  }

  // make <img> object that is intuitively draggable and zoomable
  function makeLightboxImg(sourceImg, container, numberInfoBox, zoomInfoBox) {
    // create copy of source <img>
    const img = sourceImg.cloneNode(true);
    img.classList.remove("lightbox_document_img");
    img.removeAttribute("id");
    img.removeAttribute("width");
    img.removeAttribute("height");
    img.style.position = "unset";
    img.style.margin = "0";
    img.style.padding = "0";
    img.style.width = "";
    img.style.height = "";
    img.style.minWidth = "";
    img.style.minHeight = "";
    img.style.maxWidth = "";
    img.style.maxHeight = "";
    img.id = "lightbox_img";

    // build sorted list of zoomSteps
    const zoomSteps = zooms.split(/[^0-9.]/).map((step) => parseFloat(step));
    zoomSteps.sort((a, b) => a - b);

    // <img> object property variables
    let zoom = 1;
    let translateX = 0;
    let translateY = 0;
    let clickMouseX = undefined;
    let clickMouseY = undefined;
    let clickTranslateX = undefined;
    let clickTranslateY = undefined;

    updateNumberInfo();

    // update image numbers displayed in info box
    function updateNumberInfo() {
      numberInfoBox.innerHTML =
        sourceImg.dataset.number + " of " + sourceImg.dataset.total;
    }

    // update zoom displayed in info box
    function updateZoomInfo() {
      let zoomInfo = zoom * 100;
      if (!Number.isInteger(zoomInfo)) zoomInfo = zoomInfo.toFixed(2);
      zoomInfoBox.innerHTML = zoomInfo + "%";
    }

    // move to closest zoom step above current zoom
    const zoomIn = function () {
      for (const zoomStep of zoomSteps) {
        if (zoomStep > zoom) {
          zoom = zoomStep;
          break;
        }
      }
      updateTransform();
    };

    // move to closest zoom step above current zoom
    const zoomOut = function () {
      zoomSteps.reverse();
      for (const zoomStep of zoomSteps) {
        if (zoomStep < zoom) {
          zoom = zoomStep;
          break;
        }
      }
      zoomSteps.reverse();

      updateTransform();
    };

    // update display of <img> based on scale/translate properties
    const updateTransform = function () {
      // set transform
      img.style.transform =
        "translate(" +
        (translateX || 0) +
        "px," +
        (translateY || 0) +
        "px) scale(" +
        (zoom || 1) +
        ")";

      // get new width/height after scale
      const rect = img.getBoundingClientRect();
      // limit translate
      translateX = Math.max(translateX, -rect.width / 2);
      translateX = Math.min(translateX, rect.width / 2);
      translateY = Math.max(translateY, -rect.height / 2);
      translateY = Math.min(translateY, rect.height / 2);

      // set transform
      img.style.transform =
        "translate(" +
        (translateX || 0) +
        "px," +
        (translateY || 0) +
        "px) scale(" +
        (zoom || 1) +
        ")";

      updateZoomInfo();
    };

    // fit <img> to container
    const fit = function () {
      // no x/y offset, 100% zoom by default
      translateX = 0;
      translateY = 0;
      zoom = 1;

      // widths of <img> and container
      const imgWidth = img.naturalWidth;
      const imgHeight = img.naturalHeight;
      const containerWidth = parseFloat(
        window.getComputedStyle(container).width
      );
      const containerHeight = parseFloat(
        window.getComputedStyle(container).height
      );

      // how much zooming is needed to fit <img> to container
      const xRatio = imgWidth / containerWidth;
      const yRatio = imgHeight / containerHeight;
      const maxRatio = Math.max(xRatio, yRatio);
      const newZoom = 1 / maxRatio;

      // fit <img> to container according to option
      if (defaultZoom === "shrink") {
        if (maxRatio > 1) zoom = newZoom;
      } else if (defaultZoom === "fit") zoom = newZoom;

      updateTransform();
    };

    // when mouse wheel is rolled anywhere in container
    const onContainerWheel = function (event) {
      if (!event) return;

      // let ctrl + mouse wheel to zoom behave as normal
      if (event.ctrlKey) return;

      // prevent normal scroll behavior
      event.preventDefault();
      event.stopPropagation();

      // point around which to scale img
      const viewRect = container.getBoundingClientRect();
      const viewX = (viewRect.left + viewRect.right) / 2;
      const viewY = (viewRect.top + viewRect.bottom) / 2;
      const originX = centerZoom === "true" ? viewX : mouseX;
      const originY = centerZoom === "true" ? viewY : mouseY;

      // get point on image under origin
      const oldRect = img.getBoundingClientRect();
      const oldPercentX = (originX - oldRect.left) / oldRect.width;
      const oldPercentY = (originY - oldRect.top) / oldRect.height;

      // increment/decrement zoom
      if (event.deltaY < 0) zoomIn();
      if (event.deltaY > 0) zoomOut();

      // get offset between previous image point and origin
      const newRect = img.getBoundingClientRect();
      const offsetX = originX - (newRect.left + newRect.width * oldPercentX);
      const offsetY = originY - (newRect.top + newRect.height * oldPercentY);

      // translate image to keep image point under origin
      translateX += offsetX;
      translateY += offsetY;

      // perform translate
      updateTransform();
    };

    // when container is clicked
    function onContainerClick(event) {
      // if container itself is target of click, and not other
      // element above it
      if (event.target === this) closeLightbox();
    }

    // when mouse button is pressed on image
    const onImageMouseDown = function (event) {
      // store original mouse position relative to image
      clickMouseX = window.mouseX;
      clickMouseY = window.mouseY;
      clickTranslateX = translateX;
      clickTranslateY = translateY;
      event.stopPropagation();
      event.preventDefault();
    };

    // when mouse button is released anywhere in window
    const onWindowMouseUp = function (event) {
      // reset original mouse position
      clickMouseX = undefined;
      clickMouseY = undefined;
      clickTranslateX = undefined;
      clickTranslateY = undefined;

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("mouseup", onWindowMouseUp);
    };

    // when mouse is moved anywhere in window
    const onWindowMouseMove = function (event) {
      if (
        clickMouseX === undefined ||
        clickMouseY === undefined ||
        clickTranslateX === undefined ||
        clickTranslateY === undefined
      )
        return;

      // offset image based on original and current mouse position
      translateX = clickTranslateX + window.mouseX - clickMouseX;
      translateY = clickTranslateY + window.mouseY - clickMouseY;
      updateTransform();
      event.preventDefault();

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("mousemove", onWindowMouseMove);
    };

    // when window is resized
    const onWindowResize = function (event) {
      fit();

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("resize", onWindowResize);
    };

    // attach the necessary event listeners
    img.addEventListener("dblclick", fit);
    img.addEventListener("mousedown", onImageMouseDown);
    container.addEventListener("wheel", onContainerWheel);
    container.addEventListener("mousedown", onContainerClick);
    container.addEventListener("touchstart", onContainerClick);
    window.addEventListener("mouseup", onWindowMouseUp);
    window.addEventListener("mousemove", onWindowMouseMove);
    window.addEventListener("resize", onWindowResize);

    // run fit() after lightbox atttached to document and <img> Loaded
    // so needed container and img dimensions available
    img.addEventListener("load", fit);

    return img;
  }

  // make caption
  function makeCaption(img) {
    const caption = document.createElement("div");
    caption.id = "lightbox_caption";
    const captionSource = img.nextElementSibling;
    if (captionSource.tagName.toLowerCase() === "figcaption") {
      const captionCopy = makeCopy(captionSource);
      caption.innerHTML = captionCopy.innerHTML;
    }

    caption.addEventListener("touchstart", function (event) {
      event.stopPropagation();
    });

    return caption;
  }

  // make carbon copy of html dom element
  function makeCopy(source) {
    const sourceCopy = source.cloneNode(true);

    // delete elements marked with ignore (eg anchor and jump buttons)
    const deleteFromCopy = sourceCopy.querySelectorAll('[data-ignore="true"]');
    for (const element of deleteFromCopy) element.remove();

    // delete certain element attributes
    const attributes = [
      "id",
      "data-collapsed",
      "data-selected",
      "data-highlighted",
      "data-glow",
    ];
    for (const attribute of attributes) {
      sourceCopy.removeAttribute(attribute);
      const elements = sourceCopy.querySelectorAll("[" + attribute + "]");
      for (const element of elements) element.removeAttribute(attribute);
    }

    return sourceCopy;
  }

  // make button to jump to previous image in document
  function makePrevButton(img) {
    const prevButton = document.createElement("button");
    prevButton.id = "lightbox_prev_button";
    prevButton.title = "Jump to the previous image in the document [←]";
    prevButton.classList.add("icon_button", "lightbox_button");
    prevButton.innerHTML = document.querySelector(".icon_caret_left").innerHTML;

    // attach click listeners to button
    prevButton.addEventListener("click", function () {
      getPrevImg(img).click();
    });

    return prevButton;
  }

  // make button to jump to next image in document
  function makeNextButton(img) {
    const nextButton = document.createElement("button");
    nextButton.id = "lightbox_next_button";
    nextButton.title = "Jump to the next image in the document [→]";
    nextButton.classList.add("icon_button", "lightbox_button");
    nextButton.innerHTML = document.querySelector(
      ".icon_caret_right"
    ).innerHTML;

    // attach click listeners to button
    nextButton.addEventListener("click", function () {
      getNextImg(img).click();
    });

    return nextButton;
  }

  // get previous image in document
  function getPrevImg(img) {
    const imgs = document.querySelectorAll(".lightbox_document_img");

    // find index of provided img
    let index;
    for (index = 0; index < imgs.length; index++) {
      if (imgs[index] === img) break;
    }

    // wrap index to other side if < 1
    if (index - 1 >= 0) index--;
    else index = imgs.length - 1;
    return imgs[index];
  }

  // get next image in document
  function getNextImg(img) {
    const imgs = document.querySelectorAll(".lightbox_document_img");

    // find index of provided img
    let index;
    for (index = 0; index < imgs.length; index++) {
      if (imgs[index] === img) break;
    }

    // wrap index to other side if > total
    if (index + 1 <= imgs.length - 1) index++;
    else index = 0;
    return imgs[index];
  }

  // close lightbox
  function closeLightbox() {
    focusBody();

    const lightbox = document.getElementById("lightbox_overlay");
    if (lightbox) lightbox.remove();
  }

  // make all elements behind lightbox non-focusable
  function blurBody(overlay) {
    const all = document.querySelectorAll("*");
    for (const element of all) element.tabIndex = -1;
    document.body.classList.add("body_no_scroll");
  }

  // make all elements focusable again
  function focusBody() {
    const all = document.querySelectorAll("*");
    for (const element of all) element.removeAttribute("tabIndex");
    document.body.classList.remove("body_no_scroll");
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
  <!-- modified from: https://fontawesome.com/icons/caret-left -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
    ></path>
  </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
  <!-- modified from: https://fontawesome.com/icons/caret-right -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* regular <img> in document when hovered */
    img.lightbox_document_img:hover {
      cursor: pointer;
    }

    .body_no_scroll {
      overflow: hidden !important;
    }

    /* screen overlay */
    #lightbox_overlay {
      display: flex;
      flex-direction: column;
      position: fixed;
      left: 0;
      top: 0;
      right: 0;
      bottom: 0;
      background: rgba(0, 0, 0, 0.75);
      z-index: 3;
    }

    /* middle area containing lightbox image */
    #lightbox_image_container {
      flex-grow: 1;
      display: flex;
      justify-content: center;
      align-items: center;
      overflow: hidden;
      position: relative;
      padding: 20px;
    }

    /* bottom area containing caption */
    #lightbox_bottom_container {
      display: flex;
      justify-content: center;
      align-items: center;
      height: 100px;
      min-height: 100px;
      max-height: 100px;
      background: rgba(0, 0, 0, 0.5);
    }

    /* image number info text box */
    #lightbox_number_info {
      position: absolute;
      color: #ffffff;
      font-weight: 600;
      left: 2px;
      top: 0;
      z-index: 4;
    }

    /* zoom info text box */
    #lightbox_zoom_info {
      position: absolute;
      color: #ffffff;
      font-weight: 600;
      right: 2px;
      top: 0;
      z-index: 4;
    }

    /* copy of image caption */
    #lightbox_caption {
      box-sizing: border-box;
      display: inline-block;
      width: 100%;
      max-height: 100%;
      padding: 10px 0;
      text-align: center;
      overflow-y: auto;
      color: #ffffff;
    }

    /* navigation previous/next button */
    .lightbox_button {
      width: 100px;
      height: 100%;
      min-width: 100px;
      min-height: 100%;
      color: #ffffff;
    }

    /* navigation previous/next button when hovered */
    .lightbox_button:hover {
      background: none !important;
    }

    /* navigation button icon */
    .lightbox_button > svg {
      height: 25px;
    }

    /* figure auto-number */
    #lightbox_caption > span:first-of-type {
      font-weight: bold;
      margin-right: 5px;
    }

    /* lightbox image when hovered */
    #lightbox_img:hover {
      cursor: grab;
    }

    /* lightbox image when grabbed */
    #lightbox_img:active {
      cursor: grabbing;
    }
  }

  /* when on screen < 480px wide */
  @media only screen and (max-width: 480px) {
    /* make navigation buttons skinnier on small screens to make more room for caption text */
    .lightbox_button {
      width: 50px;
      min-width: 50px;
    }
  }

  /* always hide lightbox on print */
  @media only print {
    #lightbox_overlay {
      display: none;
    }
  }
</style>
<!-- 
  Link Highlight Plugin

  Makes it such that when a user hovers or focuses a link, other links that have
  the same target will be highlighted. It also makes it such that when clicking
  a link, the target of the link (eg reference, figure, table) is briefly
  highlighted.
-->

<script type="module">
  // whether to also highlight links that go to external urls
  const externalLinks = "false";
  // whether user must click off to unhighlight instead of just
  // un-hovering
  const clickUnhighlight = "false";
  // whether to also highlight links that are unique
  const highlightUnique = "true";

  // start script
  function start() {
    const links = getLinks();
    for (const link of links) {
      // attach mouse and focus listeners to link
      link.addEventListener("mouseenter", onLinkFocus);
      link.addEventListener("focus", onLinkFocus);
      link.addEventListener("mouseleave", onLinkUnhover);
    }

    // attach click and hash change listeners to window
    window.addEventListener("click", onClick);
    window.addEventListener("touchstart", onClick);
    window.addEventListener("hashchange", onHashChange);

    // run hash change on window load in case user has navigated
    // directly to hash
    onHashChange();
  }

  // when link is focused (tabbed to) or hovered
  function onLinkFocus() {
    highlight(this);
  }

  // when link is unhovered
  function onLinkUnhover() {
    if (clickUnhighlight !== "true") unhighlightAll();
  }

  // when the mouse is clicked anywhere in window
  function onClick(event) {
    unhighlightAll();
  }

  // when hash (eg manuscript.html#introduction) changes
  function onHashChange() {
    const target = getHashTarget();
    if (target) glowElement(target);
  }

  // start glow sequence on an element
  function glowElement(element) {
    const startGlow = function () {
      onGlowEnd();
      element.dataset.glow = "true";
      element.addEventListener("animationend", onGlowEnd);
    };
    const onGlowEnd = function () {
      element.removeAttribute("data-glow");
      element.removeEventListener("animationend", onGlowEnd);
    };
    startGlow();
  }

  // highlight link and all others with same target
  function highlight(link) {
    // force unhighlight all to start fresh
    unhighlightAll();

    // get links with same target
    if (!link) return;
    const sameLinks = getSameLinks(link);

    // if link unique and option is off, exit and don't highlight
    if (sameLinks.length <= 1 && highlightUnique !== "true") return;

    // highlight all same links, and "select" (special highlight) this
    // one
    for (const sameLink of sameLinks) {
      if (sameLink === link) sameLink.setAttribute("data-selected", "true");
      else sameLink.setAttribute("data-highlighted", "true");
    }
  }

  // unhighlight all links
  function unhighlightAll() {
    const links = getLinks();
    for (const link of links) {
      link.setAttribute("data-selected", "false");
      link.setAttribute("data-highlighted", "false");
    }
  }

  // get links with same target
  function getSameLinks(link) {
    const results = [];
    const links = getLinks();
    for (const otherLink of links) {
      if (otherLink.getAttribute("href") === link.getAttribute("href"))
        results.push(otherLink);
    }
    return results;
  }

  // get all links of types we wish to handle
  function getLinks() {
    let query = "a";
    if (externalLinks !== "true") query += '[href^="#"]';
    // exclude buttons, anchor links, toc links, etc
    query += ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    return document.querySelectorAll(query);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<style>
  @media only screen {
    /* anything with data-highlighted attribute true */
    [data-highlighted="true"] {
      background: #ffeb3b;
    }

    /* anything with data-selected attribute true */
    [data-selected="true"] {
      background: #ff8a65 !important;
    }

    /* animation definition for glow */
    @keyframes highlight_glow {
      0% {
        background: none;
      }
      10% {
        background: #bbdefb;
      }
      100% {
        background: none;
      }
    }

    /* anything with data-glow attribute true */
    [data-glow="true"] {
      animation: highlight_glow 2s;
    }
  }
</style>
<!--
  Table of Contents Plugin

  Provides a "table of contents" (toc) panel on the side of the document that
  allows the user to conveniently navigate between sections of the document.
-->

<script type="module">
  // which types of elements to add links for, in "document.querySelector" format
  const typesQuery = "h1, h2, h3";
  // whether toc starts open. use 'true' or 'false', or 'auto' to
  // use 'true' behavior when screen wide enough and 'false' when not
  const startOpen = "false";
  // whether toc closes when clicking on toc link. use 'true' or
  // 'false', or 'auto' to use 'false' behavior when screen wide
  // enough and 'true' when not
  const clickClose = "auto";
  // if list item is more than this many characters, text will be
  // truncated
  const charLimit = "50";
  // whether or not to show bullets next to each toc item
  const bullets = "false";

  // start script
  function start() {
    // make toc panel and populate with entries (links to document
    // sections)
    const panel = makePanel();
    if (!panel) return;
    makeEntries(panel);
    // attach panel to document after making entries, so 'toc' heading
    // in panel isn't included in toc
    document.body.insertBefore(panel, document.body.firstChild);

    // initial panel state
    if (startOpen === "true" || (startOpen === "auto" && !isSmallScreen()))
      openPanel();
    else closePanel();

    // attach click, scroll, and hash change listeners to window
    window.addEventListener("click", onClick);
    window.addEventListener("scroll", onScroll);
    window.addEventListener("hashchange", onScroll);
    window.addEventListener("keyup", onKeyUp);
    onScroll();

    // add class to push document body down out of way of toc button
    document.body.classList.add("toc_body_nudge");
  }

  // determine if screen wide enough to fit toc panel
  function isSmallScreen() {
    // in default theme:
    // 816px = 8.5in = width of "page" (<body>) element
    // 260px = min width of toc panel (*2 for both sides of <body>)
    return window.innerWidth < 816 + 260 * 2;
  }

  // when mouse is clicked anywhere in window
  function onClick() {
    if (isSmallScreen()) closePanel();
  }

  // when window is scrolled or hash changed
  function onScroll() {
    highlightViewed();
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    // close on esc
    if (event.key === "Escape") closePanel();
  }

  // find entry of currently viewed document section in toc and highlight
  function highlightViewed() {
    const firstId = getFirstInView(typesQuery);

    // get toc entries (links), unhighlight all, then highlight viewed
    const list = document.getElementById("toc_list");
    if (!firstId || !list) return;
    const links = list.querySelectorAll("a");
    for (const link of links) link.dataset.viewing = "false";
    const link = list.querySelector('a[href="#' + firstId + '"]');
    if (!link) return;
    link.dataset.viewing = "true";
  }

  // get first or previous toc listed element in top half of view
  function getFirstInView(query) {
    // get all elements matching query and with id
    const elements = document.querySelectorAll(query);
    const elementsWithIds = [];
    for (const element of elements) {
      if (element.id) elementsWithIds.push(element);
    }

    // get first or previous element in top half of view
    for (let i = 0; i < elementsWithIds.length; i++) {
      const element = elementsWithIds[i];
      const prevElement = elementsWithIds[Math.max(0, i - 1)];
      if (element.getBoundingClientRect().top >= 0) {
        if (element.getBoundingClientRect().top < window.innerHeight / 2)
          return element.id;
        else return prevElement.id;
      }
    }
  }

  // make panel
  function makePanel() {
    // create panel
    const panel = document.createElement("div");
    panel.id = "toc_panel";
    if (bullets === "true") panel.dataset.bullets = "true";

    // create header
    const header = document.createElement("div");
    header.id = "toc_header";

    // create toc button
    const button = document.createElement("button");
    button.id = "toc_button";
    button.innerHTML = document.querySelector(".icon_th_list").innerHTML;
    button.title = "Table of Contents";
    button.classList.add("icon_button");

    // create header text
    const text = document.createElement("h4");
    text.innerHTML = "Table of Contents";

    // create container for toc list
    const list = document.createElement("div");
    list.id = "toc_list";

    // attach click listeners
    panel.addEventListener("click", onPanelClick);
    header.addEventListener("click", onHeaderClick);
    button.addEventListener("click", onButtonClick);

    // attach elements
    header.appendChild(button);
    header.appendChild(text);
    panel.appendChild(header);
    panel.appendChild(list);

    return panel;
  }

  // create toc entries (links) to each element of the specified types
  function makeEntries(panel) {
    const elements = document.querySelectorAll(typesQuery);
    for (const element of elements) {
      // do not add link if element doesn't have assigned id
      if (!element.id) continue;

      // create link/list item
      const link = document.createElement("a");
      link.classList.add("toc_link");
      switch (element.tagName.toLowerCase()) {
        case "h1":
          link.dataset.level = "1";
          break;
        case "h2":
          link.dataset.level = "2";
          break;
        case "h3":
          link.dataset.level = "3";
          break;
        case "h4":
          link.dataset.level = "4";
          break;
      }
      link.title = element.innerText;
      let text = element.innerText;
      if (text.length > charLimit) text = text.slice(0, charLimit) + "...";
      link.innerHTML = text;
      link.href = "#" + element.id;
      link.addEventListener("click", onLinkClick);

      // attach link
      panel.querySelector("#toc_list").appendChild(link);
    }
  }

  // when panel is clicked
  function onPanelClick(event) {
    // stop click from propagating to window/document and closing panel
    event.stopPropagation();
  }

  // when header itself is clicked
  function onHeaderClick(event) {
    togglePanel();
  }

  // when button is clicked
  function onButtonClick(event) {
    togglePanel();
    // stop header underneath button from also being clicked
    event.stopPropagation();
  }

  // when link is clicked
  function onLinkClick(event) {
    if (clickClose === "true" || (clickClose === "auto" && isSmallScreen()))
      closePanel();
    else openPanel();
  }

  // open panel if closed, close if opened
  function togglePanel() {
    const panel = document.getElementById("toc_panel");
    if (!panel) return;

    if (panel.dataset.open === "true") closePanel();
    else openPanel();
  }

  // open panel
  function openPanel() {
    const panel = document.getElementById("toc_panel");
    if (panel) panel.dataset.open = "true";
  }

  // close panel
  function closePanel() {
    const panel = document.getElementById("toc_panel");
    if (panel) panel.dataset.open = "false";
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- th list icon -->

<template class="icon_th_list">
  <!-- modified from: https://fontawesome.com/icons/th-list -->
  <svg width="16" height="16" viewBox="0 0 512 512" tabindex="-1">
    <path
      fill="currentColor"
      d="M96 96c0 26.51-21.49 48-48 48S0 122.51 0 96s21.49-48 48-48 48 21.49 48 48zM48 208c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm0 160c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm96-236h352c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
      tabindex="-1"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* toc panel */
    #toc_panel {
      box-sizing: border-box;
      position: fixed;
      top: 0;
      left: 0;
      background: #ffffff;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      z-index: 2;
    }

    /* toc panel when closed */
    #toc_panel[data-open="false"] {
      min-width: 60px;
      width: 60px;
      height: 60px;
      border-right: solid 1px #bdbdbd;
      border-bottom: solid 1px #bdbdbd;
    }

    /* toc panel when open */
    #toc_panel[data-open="true"] {
      min-width: 260px;
      max-width: 480px;
      /* keep panel edge consistent distance away from "page" edge */
      width: calc(((100vw - 8.5in) / 2) - 30px - 40px);
      bottom: 0;
      border-right: solid 1px #bdbdbd;
    }

    /* toc panel header */
    #toc_header {
      box-sizing: border-box;
      display: flex;
      flex-direction: row;
      align-items: center;
      height: 60px;
      margin: 0;
      padding: 20px;
    }

    /* toc panel header when hovered */
    #toc_header:hover {
      cursor: pointer;
    }

    /* toc panel header when panel open */
    #toc_panel[data-open="true"] > #toc_header {
      border-bottom: solid 1px #bdbdbd;
    }

    /* toc open/close header button */
    #toc_button {
      margin-right: 20px;
    }

    /* hide toc list and header text when closed */
    #toc_panel[data-open="false"] > #toc_header > *:not(#toc_button),
    #toc_panel[data-open="false"] > #toc_list {
      display: none;
    }

    /* toc list of entries */
    #toc_list {
      box-sizing: border-box;
      width: 100%;
      padding: 20px;
      position: absolute;
      top: calc(60px + 1px);
      bottom: 0;
      overflow: auto;
    }

    /* toc entry, link to section in document */
    .toc_link {
      display: block;
      padding: 5px;
      position: relative;
      font-weight: 600;
      text-decoration: none;
    }

    /* toc entry when hovered or when "viewed" */
    .toc_link:hover,
    .toc_link[data-viewing="true"] {
      background: #f5f5f5;
    }

    /* toc entry, level 1 indentation */
    .toc_link[data-level="1"] {
      margin-left: 0;
    }

    /* toc entry, level 2 indentation */
    .toc_link[data-level="2"] {
      margin-left: 20px;
    }

    /* toc entry, level 3 indentation */
    .toc_link[data-level="3"] {
      margin-left: 40px;
    }

    /* toc entry, level 4 indentation */
    .toc_link[data-level="4"] {
      margin-left: 60px;
    }

    /* toc entry bullets */
    #toc_panel[data-bullets="true"] .toc_link[data-level]:before {
      position: absolute;
      left: -15px;
      top: -1px;
      font-size: 1.5em;
    }

    /* toc entry, level 2 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="2"]:before {
      content: "\2022";
    }

    /* toc entry, level 3 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="3"]:before {
      content: "\25AB";
    }

    /* toc entry, level 4 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="4"]:before {
      content: "-";
    }
  }

  /* when on screen < 8.5in wide */
  @media only screen and (max-width: 8.5in) {
    /* push <body> ("page") element down to make room for toc icon */
    .toc_body_nudge {
      padding-top: 60px;
    }

    /* toc icon when panel closed and not hovered */
    #toc_panel[data-open="false"]:not(:hover) {
      background: rgba(255, 255, 255, 0.75);
    }
  }

  /* always hide toc panel on print */
  @media only print {
    #toc_panel {
      display: none;
    }
  }
</style>
<!-- 
  Tooltips Plugin

  Makes it such that when the user hovers or focuses a link to a citation or
  figure, a tooltip appears with a preview of the reference content, along with
  arrows to navigate between instances of the same reference in the document.
-->

<script type="module">
  // whether user must click off to close tooltip instead of just un-hovering
  const clickClose = "false";
  // delay (in ms) between opening and closing tooltip
  const delay = "100";

  // start script
  function start() {
    const links = getLinks();
    for (const link of links) {
      // attach hover and focus listeners to link
      link.addEventListener("mouseover", onLinkHover);
      link.addEventListener("mouseleave", onLinkUnhover);
      link.addEventListener("focus", onLinkFocus);
      link.addEventListener("touchend", onLinkTouch);
    }

    // attach mouse, key, and resize listeners to window
    window.addEventListener("mousedown", onClick);
    window.addEventListener("touchstart", onClick);
    window.addEventListener("keyup", onKeyUp);
    window.addEventListener("resize", onResize);
  }

  // when link is hovered
  function onLinkHover() {
    // function to open tooltip
    const delayOpenTooltip = function () {
      openTooltip(this);
    }.bind(this);

    // run open function after delay
    this.openTooltipTimer = window.setTimeout(delayOpenTooltip, delay);
  }

  // when mouse leaves link
  function onLinkUnhover() {
    // cancel opening tooltip
    window.clearTimeout(this.openTooltipTimer);

    // don't close on unhover if option specifies
    if (clickClose === "true") return;

    // function to close tooltip
    const delayCloseTooltip = function () {
      // if tooltip open and if mouse isn't over tooltip, close
      const tooltip = document.getElementById("tooltip");
      if (tooltip && !tooltip.matches(":hover")) closeTooltip();
    };

    // run close function after delay
    this.closeTooltipTimer = window.setTimeout(delayCloseTooltip, delay);
  }

  // when link is focused (tabbed to)
  function onLinkFocus(event) {
    openTooltip(this);
  }

  // when link is touched on touch screen
  function onLinkTouch(event) {
    // attempt to force hover state on first tap always, and trigger
    // regular link click (and navigation) on second tap
    if (event.target === document.activeElement) event.target.click();
    else {
      document.activeElement.blur();
      event.target.focus();
    }
    if (event.cancelable) event.preventDefault();
    event.stopPropagation();
    return false;
  }

  // when mouse is clicked anywhere in window
  function onClick(event) {
    closeTooltip();
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    switch (event.key) {
      // trigger click of prev button
      case "ArrowLeft":
        const prevButton = document.getElementById("tooltip_prev_button");
        if (prevButton) prevButton.click();
        break;
      // trigger click of next button
      case "ArrowRight":
        const nextButton = document.getElementById("tooltip_next_button");
        if (nextButton) nextButton.click();
        break;
      // close on esc
      case "Escape":
        closeTooltip();
        break;
    }
  }

  // when window is resized or zoomed
  function onResize() {
    closeTooltip();
  }

  // get all links of types we wish to handle
  function getLinks() {
    const queries = [];
    // exclude buttons, anchor links, toc links, etc
    const exclude =
      ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    queries.push('a[href^="#ref-"]' + exclude); // citation links
    queries.push('a[href^="#fig:"]' + exclude); // figure links
    const query = queries.join(", ");
    return document.querySelectorAll(query);
  }

  // get links with same target, get index of link in set, get total
  // same links
  function getSameLinks(link) {
    const sameLinks = [];
    const links = getLinks();
    for (const otherLink of links) {
      if (otherLink.getAttribute("href") === link.getAttribute("href"))
        sameLinks.push(otherLink);
    }

    return {
      elements: sameLinks,
      index: sameLinks.indexOf(link),
      total: sameLinks.length,
    };
  }

  // open tooltip
  function openTooltip(link) {
    // delete tooltip if it exists, start fresh
    closeTooltip();

    // make tooltip element
    const tooltip = makeTooltip(link);

    // if source couldn't be found and tooltip not made, exit
    if (!tooltip) return;

    // make navbar elements
    const navBar = makeNavBar(link);
    if (navBar) tooltip.firstElementChild.appendChild(navBar);

    // attach tooltip to page
    document.body.appendChild(tooltip);

    // position tooltip
    const position = function () {
      positionTooltip(link);
    };
    position();

    // if tooltip contains images, position again after they've loaded
    const imgs = tooltip.querySelectorAll("img");
    for (const img of imgs) img.addEventListener("load", position);
  }

  // close (delete) tooltip
  function closeTooltip() {
    const tooltip = document.getElementById("tooltip");
    if (tooltip) tooltip.remove();
  }

  // make tooltip
  function makeTooltip(link) {
    // get target element that link points to
    const source = getSource(link);

    // if source can't be found, exit
    if (!source) return;

    // create new tooltip
    const tooltip = document.createElement("div");
    tooltip.id = "tooltip";
    const tooltipContent = document.createElement("div");
    tooltipContent.id = "tooltip_content";
    tooltip.appendChild(tooltipContent);

    // make copy of source node and put in tooltip
    const sourceCopy = makeCopy(source);
    tooltipContent.appendChild(sourceCopy);

    // attach mouse event listeners
    tooltip.addEventListener("click", onTooltipClick);
    tooltip.addEventListener("mousedown", onTooltipClick);
    tooltip.addEventListener("touchstart", onTooltipClick);
    tooltip.addEventListener("mouseleave", onTooltipUnhover);

    // (for interaction with lightbox plugin)
    // transfer click on tooltip copied img to original img
    const sourceImg = source.querySelector("img");
    const sourceCopyImg = sourceCopy.querySelector("img");
    if (sourceImg && sourceCopyImg) {
      const clickImg = function () {
        sourceImg.click();
        closeTooltip();
      };
      sourceCopyImg.addEventListener("click", clickImg);
    }

    return tooltip;
  }

  // make carbon copy of html dom element
  function makeCopy(source) {
    const sourceCopy = source.cloneNode(true);

    // delete elements marked with ignore (eg anchor and jump buttons)
    const deleteFromCopy = sourceCopy.querySelectorAll('[data-ignore="true"]');
    for (const element of deleteFromCopy) element.remove();

    // delete certain element attributes
    const attributes = [
      "id",
      "data-collapsed",
      "data-selected",
      "data-highlighted",
      "data-glow",
      "class",
    ];
    for (const attribute of attributes) {
      sourceCopy.removeAttribute(attribute);
      const elements = sourceCopy.querySelectorAll("[" + attribute + "]");
      for (const element of elements) element.removeAttribute(attribute);
    }

    return sourceCopy;
  }

  // when tooltip is clicked
  function onTooltipClick(event) {
    // when user clicks on tooltip, stop click from transferring
    // outside of tooltip (eg, click off to close tooltip, or eg click
    // off to unhighlight same refs)
    event.stopPropagation();
  }

  // when tooltip is unhovered
  function onTooltipUnhover(event) {
    if (clickClose === "true") return;

    // make sure new mouse/touch/focus no longer over tooltip or any
    // element within it
    const tooltip = document.getElementById("tooltip");
    if (!tooltip) return;
    if (this.contains(event.relatedTarget)) return;

    closeTooltip();
  }

  // make nav bar to go betwen prev/next instances of same reference
  function makeNavBar(link) {
    // find other links to the same source
    const sameLinks = getSameLinks(link);

    // don't show nav bar when singular reference
    if (sameLinks.total <= 1) return;

    // find prev/next links with same target
    const prevLink = getPrevLink(link, sameLinks);
    const nextLink = getNextLink(link, sameLinks);

    // create nav bar
    const navBar = document.createElement("div");
    navBar.id = "tooltip_nav_bar";
    const text = sameLinks.index + 1 + " of " + sameLinks.total;

    // create nav bar prev/next buttons
    const prevButton = document.createElement("button");
    const nextButton = document.createElement("button");
    prevButton.id = "tooltip_prev_button";
    nextButton.id = "tooltip_next_button";
    prevButton.title =
      "Jump to the previous occurence of this item in the document [←]";
    nextButton.title =
      "Jump to the next occurence of this item in the document [→]";
    prevButton.classList.add("icon_button");
    nextButton.classList.add("icon_button");
    prevButton.innerHTML = document.querySelector(".icon_caret_left").innerHTML;
    nextButton.innerHTML =
      document.querySelector(".icon_caret_right").innerHTML;
    navBar.appendChild(prevButton);
    navBar.appendChild(document.createTextNode(text));
    navBar.appendChild(nextButton);

    // attach click listeners to buttons
    prevButton.addEventListener("click", function () {
      onPrevNextClick(link, prevLink);
    });
    nextButton.addEventListener("click", function () {
      onPrevNextClick(link, nextLink);
    });

    return navBar;
  }

  // get previous link with same target
  function getPrevLink(link, sameLinks) {
    if (!sameLinks) sameLinks = getSameLinks(link);
    // wrap index to other side if < 1
    let index;
    if (sameLinks.index - 1 >= 0) index = sameLinks.index - 1;
    else index = sameLinks.total - 1;
    return sameLinks.elements[index];
  }

  // get next link with same target
  function getNextLink(link, sameLinks) {
    if (!sameLinks) sameLinks = getSameLinks(link);
    // wrap index to other side if > total
    let index;
    if (sameLinks.index + 1 <= sameLinks.total - 1) index = sameLinks.index + 1;
    else index = 0;
    return sameLinks.elements[index];
  }

  // get element that is target of link or url hash
  function getSource(link) {
    const hash = link ? link.hash : window.location.hash;
    const id = hash.slice(1);
    let target = document.querySelector('[id="' + id + '"]');
    if (!target) return;

    // if ref or figure, modify target to get expected element
    if (id.indexOf("ref-") === 0) target = target.querySelector(":nth-child(2)");
    else if (id.indexOf("fig:") === 0) target = target.querySelector("figure");

    return target;
  }

  // when prev/next arrow button is clicked
  function onPrevNextClick(link, prevNextLink) {
    if (link && prevNextLink)
      goToElement(prevNextLink, window.innerHeight * 0.5);
  }

  // scroll to and focus element
  function goToElement(element, offset) {
    // expand accordion section if collapsed
    expandElement(element);
    const y =
      getRectInView(element).top -
      getRectInView(document.documentElement).top -
      (offset || 0);
    // trigger any function listening for "onscroll" event
    window.dispatchEvent(new Event("scroll"));
    window.scrollTo(0, y);
    document.activeElement.blur();
    element.focus();
  }

  // determine position to place tooltip based on link position in
  // viewport and tooltip size
  function positionTooltip(link, left, top) {
    const tooltipElement = document.getElementById("tooltip");
    if (!tooltipElement) return;

    // get convenient vars for position/dimensions of
    // link/tooltip/page/view
    link = getRectInPage(link);
    const tooltip = getRectInPage(tooltipElement);
    const view = getRectInPage();

    // horizontal positioning
    if (left)
      // use explicit value
      left = left;
    else if (link.left + tooltip.width < view.right)
      // fit tooltip to right of link
      left = link.left;
    else if (link.right - tooltip.width > view.left)
      // fit tooltip to left of link
      left = link.right - tooltip.width;
    // center tooltip in view
    else left = (view.right - view.left) / 2 - tooltip.width / 2;

    // vertical positioning
    if (top)
      // use explicit value
      top = top;
    else if (link.top - tooltip.height > view.top)
      // fit tooltip above link
      top = link.top - tooltip.height;
    else if (link.bottom + tooltip.height < view.bottom)
      // fit tooltip below link
      top = link.bottom;
    else {
      // center tooltip in view
      top = view.top + view.height / 2 - tooltip.height / 2;
      // nudge off of link to left/right if possible
      if (link.right + tooltip.width < view.right) left = link.right;
      else if (link.left - tooltip.width > view.left)
        left = link.left - tooltip.width;
    }

    tooltipElement.style.left = left + "px";
    tooltipElement.style.top = top + "px";
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
  <!-- modified from: https://fontawesome.com/icons/caret-left -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
    ></path>
  </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
  <!-- modified from: https://fontawesome.com/icons/caret-right -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* tooltip container */
    #tooltip {
      position: absolute;
      width: 50%;
      min-width: 240px;
      max-width: 75%;
      z-index: 1;
    }

    /* tooltip content */
    #tooltip_content {
      margin-bottom: 5px;
      padding: 20px;
      border-radius: 5px;
      border: solid 1px #bdbdbd;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      background: #ffffff;
      overflow-wrap: break-word;
    }

    /* tooltip copy of paragraphs and figures */
    #tooltip_content > p,
    #tooltip_content > figure {
      margin: 0;
      max-height: 320px;
      overflow-y: auto;
    }

    /* tooltip copy of <img> */
    #tooltip_content > figure > img,
    #tooltip_content > figure > svg {
      max-height: 260px;
    }

    /* navigation bar */
    #tooltip_nav_bar {
      margin-top: 10px;
      text-align: center;
    }

    /* navigation bar previous/next buton */
    #tooltip_nav_bar > .icon_button {
      position: relative;
      top: 3px;
    }

    /* navigation bar previous button */
    #tooltip_nav_bar > .icon_button:first-of-type {
      margin-right: 5px;
    }

    /* navigation bar next button */
    #tooltip_nav_bar > .icon_button:last-of-type {
      margin-left: 5px;
    }
  }

  /* always hide tooltip on print */
  @media only print {
    #tooltip {
      display: none;
    }
  }
</style>
<!--
  Analytics Plugin (third-party) 
  
  Copy and paste code from Google Analytics or similar service here.
-->
<!-- 
  Annotations Plugin

  Allows public annotation of the  manuscript. See https://web.hypothes.is/.
-->

<script type="module">
  // configuration
  window.hypothesisConfig = function () {
    return {
      branding: {
        accentColor: "#2196f3",
        appBackgroundColor: "#f8f8f8",
        ctaBackgroundColor: "#f8f8f8",
        ctaTextColor: "#000000",
        selectionFontFamily: "Open Sans, Helvetica, sans serif",
        annotationFontFamily: "Open Sans, Helvetica, sans serif",
      },
    };
  };

  // hypothesis client script
  const embed = "https://hypothes.is/embed.js";
  // hypothesis annotation count query url
  const query = "https://api.hypothes.is/api/search?limit=0&url=";

  // start script
  function start() {
    const button = makeButton();
    document.body.insertBefore(button, document.body.firstChild);
    insertCount(button);
  }

  // make button
  function makeButton() {
    // create button
    const button = document.createElement("button");
    button.id = "hypothesis_button";
    button.innerHTML = document.querySelector(".icon_hypothesis").innerHTML;
    button.title = "Hypothesis annotations";
    button.classList.add("icon_button");

    function onClick(event) {
      onButtonClick(event, button);
    }

    // attach click listeners
    button.addEventListener("click", onClick);

    return button;
  }

  // insert annotations count
  async function insertCount(button) {
    // get annotation count from Hypothesis based on url
    let count = "-";
    try {
      const canonical = document.querySelector('link[rel="canonical"]');
      const location = window.location;
      const url = encodeURIComponent((canonical || location).href);
      const response = await fetch(query + url);
      const json = await response.json();
      count = json.total || "-";
    } catch (error) {
      console.log(error);
    }

    // put count into button
    const counter = document.createElement("span");
    counter.id = "hypothesis_count";
    counter.innerHTML = count;
    button.title = "View " + count + " Hypothesis annotations";
    button.append(counter);
  }

  // when button is clicked
  function onButtonClick(event, button) {
    const script = document.createElement("script");
    script.src = embed;
    document.body.append(script);
    button.remove();
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- hypothesis icon -->

<template class="icon_hypothesis">
  <!-- modified from: https://simpleicons.org/icons/hypothesis.svg / https://git.io/Jf1VB -->
  <svg width="16" height="16" viewBox="0 0 24 24" tabindex="-1">
    <path
      fill="currentColor"
      d="M3.43 0C2.5 0 1.72 .768 1.72 1.72V18.86C1.72 19.8 2.5 20.57 3.43 20.57H9.38L12 24L14.62 20.57H20.57C21.5 20.57 22.29 19.8 22.29 18.86V1.72C22.29 .77 21.5 0 20.57 0H3.43M5.14 3.43H7.72V9.43S8.58 7.72 10.28 7.72C12 7.72 13.74 8.57 13.74 11.24V17.14H11.16V12C11.16 10.61 10.28 10.07 9.43 10.29C8.57 10.5 7.72 11.41 7.72 13.29V17.14H5.14V3.43M18 13.72C18.95 13.72 19.72 14.5 19.72 15.42A1.71 1.71 0 0 1 18 17.13A1.71 1.71 0 0 1 16.29 15.42C16.29 14.5 17.05 13.71 18 13.71Z"
      tabindex="-1"
    ></path>
  </svg>
</template>

<style>
  /* hypothesis activation button */
  #hypothesis_button {
    box-sizing: border-box;
    position: fixed;
    top: 0;
    right: 0;
    width: 60px;
    height: 60px;
    background: #ffffff;
    border-radius: 0;
    border-left: solid 1px #bdbdbd;
    border-bottom: solid 1px #bdbdbd;
    box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
    z-index: 2;
  }

  /* hypothesis button svg */
  #hypothesis_button > svg {
    position: relative;
    top: -4px;
  }

  /* hypothesis annotation count */
  #hypothesis_count {
    position: absolute;
    left: 0;
    right: 0;
    bottom: 5px;
  }

  /* side panel */
  .annotator-frame {
    width: 280px !important;
  }

  /* match highlight color to rest of theme */
  .annotator-highlights-always-on .annotator-hl {
    background-color: #ffeb3b !important;
  }

  /* match focused color to rest of theme */
  .annotator-hl.annotator-hl-focused {
    background-color: #ff8a65 !important;
  }

  /* match bucket bar color to rest of theme */
  .annotator-bucket-bar {
    background: #f5f5f5 !important;
  }

  /* always hide button, toolbar, and tooltip on print */
  @media only print {
    #hypothesis_button {
      display: none;
    }

    .annotator-frame {
      display: none !important;
    }

    hypothesis-adder {
      display: none !important;
    }
  }
</style>
<!-- 
  Mathjax Plugin (third-party) 

  Allows the proper rendering of math/equations written in LaTeX.
  See https://www.mathjax.org/.
-->

<script type="text/x-mathjax-config">
  // configuration
  MathJax.Hub.Config({
    "CommonHTML": { linebreaks: { automatic: true } },
    "HTML-CSS": { linebreaks: { automatic: true } },
    "SVG": { linebreaks: { automatic: true } },
    "fast-preview": { disabled: true }
  });
</script>

<script
  src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"
  integrity="sha512-0doc9hKxR3PYwso42RD1p5ySZpzzuDiOwMrdCEh2WdJZCjcmFKc/wEnL+z8fBQrnHoiNWbo+3fiGkOYXBdQp4A=="
  crossorigin="anonymous"
></script>

<style>
  /* mathjax containers */
  .math.display > span:not(.MathJax_Preview) {
    /* turn inline element (no dimensions) into block (allows fixed width and thus scrolling) */
    display: flex !important;
    overflow-x: auto !important;
    overflow-y: hidden !important;
    justify-content: center;
    align-items: center;
    margin: 0 !important;
  }

  /* right click menu */
  .MathJax_Menu {
    border-radius: 5px !important;
    border: solid 1px #bdbdbd !important;
    box-shadow: none !important;
  }

  /* equation auto-number */
  span[id^="eq:"] > span.math.display + span {
    font-weight: 600;
  }

  /* equation */
  span[id^="eq:"] > span.math.display > span {
    /* nudge to make room for equation auto-number and anchor */
    margin-right: 60px !important;
  }
</style>
</body>
</html>
